2014/04/26 17:48:37 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'The requested page does not exist.' in C:\AppServ\www\epretest\protected\controllers\web\ExamController.php:208
Stack trace:
#0 C:\AppServ\www\epretest\protected\controllers\web\ExamController.php(226): ExamController->loadTestRecord(NULL)
#1 C:\AppServ\www\epretest\protected\controllers\web\ExamController.php(113): ExamController->saveExamForm()
#2 C:\AppServ\www\epretest\framework\web\actions\CInlineAction.php(50): ExamController->actionSave()
#3 C:\AppServ\www\epretest\framework\web\CController.php(309): CInlineAction->runWithParams(Array)
#4 C:\AppServ\www\epretest\framework\web\CController.php(287): CController->runAction(Object(CInlineAction))
#5 C:\AppServ\www\epretest\framework\web\CController.php(266): CController->runActionWithFilters(Object(CInlineAction), Array)
#6 C:\AppServ\www\epretest\framework\web\CWebApplication.php(276): CController->run('save')
#7 C:\AppServ\www\epretest\framework\web\CWebApplication.php(135): CWebApplication->runController('exam/save')
#8 C:\AppServ\www\epretest\framework\base\CApplication.php(162): CWebApplication->processRequest()
#9 C:\AppServ\www\epretest\protected\components\WebApplicationEndBehavior.php(26): CApplication->run()
#10 [internal function]: WebApplicationEndBehavior->runEnd('web')
#11 C:\AppServ\www\epretest\framework\base\CComponent.php(261): call_user_func_array(Array, Array)
#12 [internal function]: CComponent->__call('runEnd', Array)
#13 C:\AppServ\www\epretest\index.php(11): CWebApplication->runEnd('web')
#14 {main}
REQUEST_URI=/epretest/index.php?r=exam/save
---
2014/04/26 17:49:02 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'The requested page does not exist.' in C:\AppServ\www\epretest\protected\controllers\web\ExamController.php:208
Stack trace:
#0 C:\AppServ\www\epretest\protected\controllers\web\ExamController.php(226): ExamController->loadTestRecord(NULL)
#1 C:\AppServ\www\epretest\protected\controllers\web\ExamController.php(113): ExamController->saveExamForm()
#2 C:\AppServ\www\epretest\framework\web\actions\CInlineAction.php(50): ExamController->actionSave()
#3 C:\AppServ\www\epretest\framework\web\CController.php(309): CInlineAction->runWithParams(Array)
#4 C:\AppServ\www\epretest\framework\web\CController.php(287): CController->runAction(Object(CInlineAction))
#5 C:\AppServ\www\epretest\framework\web\CController.php(266): CController->runActionWithFilters(Object(CInlineAction), Array)
#6 C:\AppServ\www\epretest\framework\web\CWebApplication.php(276): CController->run('save')
#7 C:\AppServ\www\epretest\framework\web\CWebApplication.php(135): CWebApplication->runController('exam/save')
#8 C:\AppServ\www\epretest\framework\base\CApplication.php(162): CWebApplication->processRequest()
#9 C:\AppServ\www\epretest\protected\components\WebApplicationEndBehavior.php(26): CApplication->run()
#10 [internal function]: WebApplicationEndBehavior->runEnd('web')
#11 C:\AppServ\www\epretest\framework\base\CComponent.php(261): call_user_func_array(Array, Array)
#12 [internal function]: CComponent->__call('runEnd', Array)
#13 C:\AppServ\www\epretest\index.php(11): CWebApplication->runEnd('web')
#14 {main}
REQUEST_URI=/epretest/index.php?r=exam/save
---
2014/04/26 18:14:12 [error] [exception.CException] exception 'CException' with message 'ExamController and its behaviors do not have a method or closure named "SevaTesting".' in C:\AppServ\www\epretest\framework\base\CComponent.php:266
Stack trace:
#0 [internal function]: CComponent->__call('SevaTesting', Array)
#1 C:\AppServ\www\epretest\protected\controllers\web\ExamController.php(416): ExamController->SevaTesting('14', '8', Array, Array)
#2 C:\AppServ\www\epretest\protected\controllers\web\ExamController.php(115): ExamController->saveExamForm()
#3 C:\AppServ\www\epretest\framework\web\actions\CInlineAction.php(50): ExamController->actionSave()
#4 C:\AppServ\www\epretest\framework\web\CController.php(309): CInlineAction->runWithParams(Array)
#5 C:\AppServ\www\epretest\framework\web\CController.php(287): CController->runAction(Object(CInlineAction))
#6 C:\AppServ\www\epretest\framework\web\CController.php(266): CController->runActionWithFilters(Object(CInlineAction), Array)
#7 C:\AppServ\www\epretest\framework\web\CWebApplication.php(276): CController->run('save')
#8 C:\AppServ\www\epretest\framework\web\CWebApplication.php(135): CWebApplication->runController('exam/save')
#9 C:\AppServ\www\epretest\framework\base\CApplication.php(162): CWebApplication->processRequest()
#10 C:\AppServ\www\epretest\protected\components\WebApplicationEndBehavior.php(26): CApplication->run()
#11 [internal function]: WebApplicationEndBehavior->runEnd('web')
#12 C:\AppServ\www\epretest\framework\base\CComponent.php(261): call_user_func_array(Array, Array)
#13 [internal function]: CComponent->__call('runEnd', Array)
#14 C:\AppServ\www\epretest\index.php(11): CWebApplication->runEnd('web')
#15 {main}
REQUEST_URI=/epretest/index.php?r=exam/save
HTTP_REFERER=http://localhost/epretest/index.php?r=exam&id=320b$14$3$5t97%3C7171
---
2014/04/26 23:35:11 [error] [system.db.CDbCommand] CDbCommand::fetchAll() failed: SQLSTATE[42S02]: Base table or view not found: 1146 Table 'epretest.esto_type' doesn't exist. The SQL statement executed was: SHOW COLUMNS FROM `esto_type`.
in C:\AppServ\www\epretest\protected\models\Type.php (24)
in C:\AppServ\www\epretest\protected\views\web\site\index.php (99)
in C:\AppServ\www\epretest\protected\controllers\web\SiteController.php (81)
2014/04/26 23:35:11 [error] [exception.CDbException] exception 'CDbException' with message 'The table "esto_type" for active record class "Type" cannot be found in the database.' in C:\AppServ\www\epretest\framework\db\ar\CActiveRecord.php:2264
Stack trace:
#0 C:\AppServ\www\epretest\framework\db\ar\CActiveRecord.php(379): CActiveRecordMetaData->__construct(Object(Type))
#1 C:\AppServ\www\epretest\protected\models\Type.php(24): CActiveRecord::model('Type')
#2 C:\AppServ\www\epretest\protected\views\web\site\index.php(99): Type::model()
#3 C:\AppServ\www\epretest\framework\web\CBaseController.php(127): require('C:\AppServ\www\...')
#4 C:\AppServ\www\epretest\framework\web\CBaseController.php(96): CBaseController->renderInternal('C:\AppServ\www\...', Array, true)
#5 C:\AppServ\www\epretest\framework\web\CController.php(870): CBaseController->renderFile('C:\AppServ\www\...', Array, true)
#6 C:\AppServ\www\epretest\framework\web\CController.php(783): CController->renderPartial('index', Array, true)
#7 C:\AppServ\www\epretest\protected\controllers\web\SiteController.php(81): CController->render('index', Array)
#8 C:\AppServ\www\epretest\framework\web\actions\CInlineAction.php(50): SiteController->actionIndex()
#9 C:\AppServ\www\epretest\framework\web\CController.php(309): CInlineAction->runWithParams(Array)
#10 C:\AppServ\www\epretest\framework\web\CController.php(287): CController->runAction(Object(CInlineAction))
#11 C:\AppServ\www\epretest\framework\web\CController.php(266): CController->runActionWithFilters(Object(CInlineAction), Array)
#12 C:\AppServ\www\epretest\framework\web\CWebApplication.php(276): CController->run('')
#13 C:\AppServ\www\epretest\framework\web\CWebApplication.php(135): CWebApplication->runController('')
#14 C:\AppServ\www\epretest\framework\base\CApplication.php(162): CWebApplication->processRequest()
#15 C:\AppServ\www\epretest\protected\components\WebApplicationEndBehavior.php(26): CApplication->run()
#16 [internal function]: WebApplicationEndBehavior->runEnd('web')
#17 C:\AppServ\www\epretest\framework\base\CComponent.php(261): call_user_func_array(Array, Array)
#18 [internal function]: CComponent->__call('runEnd', Array)
#19 C:\AppServ\www\epretest\index.php(11): CWebApplication->runEnd('web')
#20 {main}
REQUEST_URI=/epretest/
---
2014/04/27 17:33:21 [warning] [application] Failed to set unsafe attribute "hddnValue" of "LoginForm".
in C:\AppServ\www\epretest\protected\controllers\web\SiteController.php (265)
in C:\AppServ\www\epretest\protected\components\WebApplicationEndBehavior.php (26)
in C:\AppServ\www\epretest\index.php (11)
2014/04/28 18:49:35 [warning] [application] Failed to set unsafe attribute "hddnValue" of "LoginForm".
in C:\AppServ\www\epretest\protected\controllers\web\SiteController.php (265)
in C:\AppServ\www\epretest\protected\components\WebApplicationEndBehavior.php (26)
in C:\AppServ\www\epretest\index.php (11)
2014/04/28 18:49:49 [error] [exception.CException] exception 'CException' with message 'ExamController cannot find the requested view "index".' in C:\AppServ\www\epretest\framework\web\CController.php:879
Stack trace:
#0 C:\AppServ\www\epretest\framework\web\CController.php(783): CController->renderPartial('index', Array, true)
#1 C:\AppServ\www\epretest\protected\controllers\web\ExamController.php(45): CController->render('index', Array)
#2 C:\AppServ\www\epretest\framework\web\actions\CInlineAction.php(50): ExamController->actionIndex()
#3 C:\AppServ\www\epretest\framework\web\CController.php(309): CInlineAction->runWithParams(Array)
#4 C:\AppServ\www\epretest\framework\web\CController.php(287): CController->runAction(Object(CInlineAction))
#5 C:\AppServ\www\epretest\framework\web\CController.php(266): CController->runActionWithFilters(Object(CInlineAction), Array)
#6 C:\AppServ\www\epretest\framework\web\CWebApplication.php(276): CController->run('')
#7 C:\AppServ\www\epretest\framework\web\CWebApplication.php(135): CWebApplication->runController('exam')
#8 C:\AppServ\www\epretest\framework\base\CApplication.php(162): CWebApplication->processRequest()
#9 C:\AppServ\www\epretest\protected\components\WebApplicationEndBehavior.php(26): CApplication->run()
#10 [internal function]: WebApplicationEndBehavior->runEnd('web')
#11 C:\AppServ\www\epretest\framework\base\CComponent.php(261): call_user_func_array(Array, Array)
#12 [internal function]: CComponent->__call('runEnd', Array)
#13 C:\AppServ\www\epretest\index.php(11): CWebApplication->runEnd('web')
#14 {main}
REQUEST_URI=/epretest/index.php?r=exam&id=320b$7$3$5t97%3C5869
HTTP_REFERER=http://localhost/epretest/index.php?r=student/view&level=13
---
2014/04/28 18:50:07 [error] [exception.CException] exception 'CException' with message 'ExamController cannot find the requested view "index".' in C:\AppServ\www\epretest\framework\web\CController.php:879
Stack trace:
#0 C:\AppServ\www\epretest\framework\web\CController.php(783): CController->renderPartial('index', Array, true)
#1 C:\AppServ\www\epretest\protected\controllers\web\ExamController.php(45): CController->render('index', Array)
#2 C:\AppServ\www\epretest\framework\web\actions\CInlineAction.php(50): ExamController->actionIndex()
#3 C:\AppServ\www\epretest\framework\web\CController.php(309): CInlineAction->runWithParams(Array)
#4 C:\AppServ\www\epretest\framework\web\CController.php(287): CController->runAction(Object(CInlineAction))
#5 C:\AppServ\www\epretest\framework\web\CController.php(266): CController->runActionWithFilters(Object(CInlineAction), Array)
#6 C:\AppServ\www\epretest\framework\web\CWebApplication.php(276): CController->run('')
#7 C:\AppServ\www\epretest\framework\web\CWebApplication.php(135): CWebApplication->runController('exam')
#8 C:\AppServ\www\epretest\framework\base\CApplication.php(162): CWebApplication->processRequest()
#9 C:\AppServ\www\epretest\protected\components\WebApplicationEndBehavior.php(26): CApplication->run()
#10 [internal function]: WebApplicationEndBehavior->runEnd('web')
#11 C:\AppServ\www\epretest\framework\base\CComponent.php(261): call_user_func_array(Array, Array)
#12 [internal function]: CComponent->__call('runEnd', Array)
#13 C:\AppServ\www\epretest\index.php(11): CWebApplication->runEnd('web')
#14 {main}
REQUEST_URI=/epretest/index.php?r=exam&id=320b$7$3$5t97%3C5469
HTTP_REFERER=http://localhost/epretest/index.php?r=student/view&level=13
---
2014/04/28 18:50:25 [error] [exception.CException] exception 'CException' with message 'ExamController cannot find the requested view "index".' in C:\AppServ\www\epretest\framework\web\CController.php:879
Stack trace:
#0 C:\AppServ\www\epretest\framework\web\CController.php(783): CController->renderPartial('index', Array, true)
#1 C:\AppServ\www\epretest\protected\controllers\web\ExamController.php(45): CController->render('index', Array)
#2 C:\AppServ\www\epretest\framework\web\actions\CInlineAction.php(50): ExamController->actionIndex()
#3 C:\AppServ\www\epretest\framework\web\CController.php(309): CInlineAction->runWithParams(Array)
#4 C:\AppServ\www\epretest\framework\web\CController.php(287): CController->runAction(Object(CInlineAction))
#5 C:\AppServ\www\epretest\framework\web\CController.php(266): CController->runActionWithFilters(Object(CInlineAction), Array)
#6 C:\AppServ\www\epretest\framework\web\CWebApplication.php(276): CController->run('')
#7 C:\AppServ\www\epretest\framework\web\CWebApplication.php(135): CWebApplication->runController('exam')
#8 C:\AppServ\www\epretest\framework\base\CApplication.php(162): CWebApplication->processRequest()
#9 C:\AppServ\www\epretest\protected\components\WebApplicationEndBehavior.php(26): CApplication->run()
#10 [internal function]: WebApplicationEndBehavior->runEnd('web')
#11 C:\AppServ\www\epretest\framework\base\CComponent.php(261): call_user_func_array(Array, Array)
#12 [internal function]: CComponent->__call('runEnd', Array)
#13 C:\AppServ\www\epretest\index.php(11): CWebApplication->runEnd('web')
#14 {main}
REQUEST_URI=/epretest/index.php?r=exam&id=320b$30$3$5t97%3C5469
---
2014/04/28 18:50:48 [error] [exception.CException] exception 'CException' with message 'ExamController cannot find the requested view "index".' in C:\AppServ\www\epretest\framework\web\CController.php:879
Stack trace:
#0 C:\AppServ\www\epretest\framework\web\CController.php(783): CController->renderPartial('index', Array, true)
#1 C:\AppServ\www\epretest\protected\controllers\web\ExamController.php(45): CController->render('index', Array)
#2 C:\AppServ\www\epretest\framework\web\actions\CInlineAction.php(50): ExamController->actionIndex()
#3 C:\AppServ\www\epretest\framework\web\CController.php(309): CInlineAction->runWithParams(Array)
#4 C:\AppServ\www\epretest\framework\web\CController.php(287): CController->runAction(Object(CInlineAction))
#5 C:\AppServ\www\epretest\framework\web\CController.php(266): CController->runActionWithFilters(Object(CInlineAction), Array)
#6 C:\AppServ\www\epretest\framework\web\CWebApplication.php(276): CController->run('')
#7 C:\AppServ\www\epretest\framework\web\CWebApplication.php(135): CWebApplication->runController('exam')
#8 C:\AppServ\www\epretest\framework\base\CApplication.php(162): CWebApplication->processRequest()
#9 C:\AppServ\www\epretest\protected\components\WebApplicationEndBehavior.php(26): CApplication->run()
#10 [internal function]: WebApplicationEndBehavior->runEnd('web')
#11 C:\AppServ\www\epretest\framework\base\CComponent.php(261): call_user_func_array(Array, Array)
#12 [internal function]: CComponent->__call('runEnd', Array)
#13 C:\AppServ\www\epretest\index.php(11): CWebApplication->runEnd('web')
#14 {main}
REQUEST_URI=/epretest/index.php?r=exam&id=320b$7$3$5t97%3C5469
HTTP_REFERER=http://localhost/epretest/index.php?r=student/view&level=13
---
2014/04/28 18:53:06 [error] [exception.CException] exception 'CException' with message 'ExamController cannot find the requested view "index".' in C:\AppServ\www\epretest\framework\web\CController.php:879
Stack trace:
#0 C:\AppServ\www\epretest\framework\web\CController.php(783): CController->renderPartial('index', Array, true)
#1 C:\AppServ\www\epretest\protected\controllers\web\ExamController.php(45): CController->render('index', Array)
#2 C:\AppServ\www\epretest\framework\web\actions\CInlineAction.php(50): ExamController->actionIndex()
#3 C:\AppServ\www\epretest\framework\web\CController.php(309): CInlineAction->runWithParams(Array)
#4 C:\AppServ\www\epretest\framework\web\CController.php(287): CController->runAction(Object(CInlineAction))
#5 C:\AppServ\www\epretest\framework\web\CController.php(266): CController->runActionWithFilters(Object(CInlineAction), Array)
#6 C:\AppServ\www\epretest\framework\web\CWebApplication.php(276): CController->run('')
#7 C:\AppServ\www\epretest\framework\web\CWebApplication.php(135): CWebApplication->runController('exam')
#8 C:\AppServ\www\epretest\framework\base\CApplication.php(162): CWebApplication->processRequest()
#9 C:\AppServ\www\epretest\protected\components\WebApplicationEndBehavior.php(26): CApplication->run()
#10 [internal function]: WebApplicationEndBehavior->runEnd('web')
#11 C:\AppServ\www\epretest\framework\base\CComponent.php(261): call_user_func_array(Array, Array)
#12 [internal function]: CComponent->__call('runEnd', Array)
#13 C:\AppServ\www\epretest\index.php(11): CWebApplication->runEnd('web')
#14 {main}
REQUEST_URI=/epretest/index.php?r=exam&id=320b$25$3$5t97%3C4045
HTTP_REFERER=http://localhost/epretest/index.php?r=student/view&level=13
---
2014/04/29 17:34:52 [warning] [application] Failed to set unsafe attribute "hddnValue" of "LoginForm".
in C:\AppServ\www\epretest\protected\controllers\web\SiteController.php (265)
in C:\AppServ\www\epretest\protected\components\WebApplicationEndBehavior.php (26)
in C:\AppServ\www\epretest\index.php (11)
2014/04/29 17:36:14 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'The requested page does not exist.' in C:\AppServ\www\epretest\protected\controllers\web\ExamController.php:215
Stack trace:
#0 C:\AppServ\www\epretest\protected\controllers\web\ExamController.php(233): ExamController->loadTestRecord(NULL)
#1 C:\AppServ\www\epretest\protected\controllers\web\ExamController.php(128): ExamController->saveExamForm()
#2 C:\AppServ\www\epretest\framework\web\actions\CInlineAction.php(50): ExamController->actionSubmit()
#3 C:\AppServ\www\epretest\framework\web\CController.php(309): CInlineAction->runWithParams(Array)
#4 C:\AppServ\www\epretest\framework\web\CController.php(287): CController->runAction(Object(CInlineAction))
#5 C:\AppServ\www\epretest\framework\web\CController.php(266): CController->runActionWithFilters(Object(CInlineAction), Array)
#6 C:\AppServ\www\epretest\framework\web\CWebApplication.php(276): CController->run('submit')
#7 C:\AppServ\www\epretest\framework\web\CWebApplication.php(135): CWebApplication->runController('exam/submit')
#8 C:\AppServ\www\epretest\framework\base\CApplication.php(162): CWebApplication->processRequest()
#9 C:\AppServ\www\epretest\protected\components\WebApplicationEndBehavior.php(26): CApplication->run()
#10 [internal function]: WebApplicationEndBehavior->runEnd('web')
#11 C:\AppServ\www\epretest\framework\base\CComponent.php(261): call_user_func_array(Array, Array)
#12 [internal function]: CComponent->__call('runEnd', Array)
#13 C:\AppServ\www\epretest\index.php(11): CWebApplication->runEnd('web')
#14 {main}
REQUEST_URI=/epretest/index.php?r=exam/submit&id=7
HTTP_REFERER=http://localhost/epretest/index.php?r=exam&id=320b$7$3$5t97%3C30756
---
2014/04/29 17:44:04 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'The requested page does not exist.' in C:\AppServ\www\epretest\protected\controllers\web\ExamController.php:215
Stack trace:
#0 C:\AppServ\www\epretest\protected\controllers\web\ExamController.php(233): ExamController->loadTestRecord(NULL)
#1 C:\AppServ\www\epretest\protected\controllers\web\ExamController.php(128): ExamController->saveExamForm()
#2 C:\AppServ\www\epretest\framework\web\actions\CInlineAction.php(50): ExamController->actionSubmit()
#3 C:\AppServ\www\epretest\framework\web\CController.php(309): CInlineAction->runWithParams(Array)
#4 C:\AppServ\www\epretest\framework\web\CController.php(287): CController->runAction(Object(CInlineAction))
#5 C:\AppServ\www\epretest\framework\web\CController.php(266): CController->runActionWithFilters(Object(CInlineAction), Array)
#6 C:\AppServ\www\epretest\framework\web\CWebApplication.php(276): CController->run('submit')
#7 C:\AppServ\www\epretest\framework\web\CWebApplication.php(135): CWebApplication->runController('exam/submit')
#8 C:\AppServ\www\epretest\framework\base\CApplication.php(162): CWebApplication->processRequest()
#9 C:\AppServ\www\epretest\protected\components\WebApplicationEndBehavior.php(26): CApplication->run()
#10 [internal function]: WebApplicationEndBehavior->runEnd('web')
#11 C:\AppServ\www\epretest\framework\base\CComponent.php(261): call_user_func_array(Array, Array)
#12 [internal function]: CComponent->__call('runEnd', Array)
#13 C:\AppServ\www\epretest\index.php(11): CWebApplication->runEnd('web')
#14 {main}
REQUEST_URI=/epretest/index.php?r=exam/submit&id=7
HTTP_REFERER=http://localhost/epretest/index.php?r=exam&id=320b$7$3$5t97%3C30756
---
2014/04/29 20:37:19 [warning] [application] Failed to set unsafe attribute "hddnValue" of "LoginForm".
in C:\AppServ\www\epretest\protected\controllers\web\SiteController.php (265)
in C:\AppServ\www\epretest\protected\components\WebApplicationEndBehavior.php (26)
in C:\AppServ\www\epretest\index.php (11)
2014/04/29 20:37:38 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'The requested page does not exist.' in C:\AppServ\www\epretest\protected\controllers\web\ExamController.php:215
Stack trace:
#0 C:\AppServ\www\epretest\protected\controllers\web\ExamController.php(233): ExamController->loadTestRecord(NULL)
#1 C:\AppServ\www\epretest\protected\controllers\web\ExamController.php(117): ExamController->saveExamForm()
#2 C:\AppServ\www\epretest\framework\web\actions\CInlineAction.php(50): ExamController->actionSave()
#3 C:\AppServ\www\epretest\framework\web\CController.php(309): CInlineAction->runWithParams(Array)
#4 C:\AppServ\www\epretest\framework\web\CController.php(287): CController->runAction(Object(CInlineAction))
#5 C:\AppServ\www\epretest\framework\web\CController.php(266): CController->runActionWithFilters(Object(CInlineAction), Array)
#6 C:\AppServ\www\epretest\framework\web\CWebApplication.php(276): CController->run('save')
#7 C:\AppServ\www\epretest\framework\web\CWebApplication.php(135): CWebApplication->runController('exam/save')
#8 C:\AppServ\www\epretest\framework\base\CApplication.php(162): CWebApplication->processRequest()
#9 C:\AppServ\www\epretest\protected\components\WebApplicationEndBehavior.php(26): CApplication->run()
#10 [internal function]: WebApplicationEndBehavior->runEnd('web')
#11 C:\AppServ\www\epretest\framework\base\CComponent.php(261): call_user_func_array(Array, Array)
#12 [internal function]: CComponent->__call('runEnd', Array)
#13 C:\AppServ\www\epretest\index.php(11): CWebApplication->runEnd('web')
#14 {main}
REQUEST_URI=/epretest/index.php?r=exam/save
HTTP_REFERER=http://localhost/epretest/index.php?r=exam&id=320b$7$3$5t97%3C22403
---
2014/04/29 20:37:50 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'The requested page does not exist.' in C:\AppServ\www\epretest\protected\controllers\web\ExamController.php:215
Stack trace:
#0 C:\AppServ\www\epretest\protected\controllers\web\ExamController.php(233): ExamController->loadTestRecord(NULL)
#1 C:\AppServ\www\epretest\protected\controllers\web\ExamController.php(117): ExamController->saveExamForm()
#2 C:\AppServ\www\epretest\framework\web\actions\CInlineAction.php(50): ExamController->actionSave()
#3 C:\AppServ\www\epretest\framework\web\CController.php(309): CInlineAction->runWithParams(Array)
#4 C:\AppServ\www\epretest\framework\web\CController.php(287): CController->runAction(Object(CInlineAction))
#5 C:\AppServ\www\epretest\framework\web\CController.php(266): CController->runActionWithFilters(Object(CInlineAction), Array)
#6 C:\AppServ\www\epretest\framework\web\CWebApplication.php(276): CController->run('save')
#7 C:\AppServ\www\epretest\framework\web\CWebApplication.php(135): CWebApplication->runController('exam/save')
#8 C:\AppServ\www\epretest\framework\base\CApplication.php(162): CWebApplication->processRequest()
#9 C:\AppServ\www\epretest\protected\components\WebApplicationEndBehavior.php(26): CApplication->run()
#10 [internal function]: WebApplicationEndBehavior->runEnd('web')
#11 C:\AppServ\www\epretest\framework\base\CComponent.php(261): call_user_func_array(Array, Array)
#12 [internal function]: CComponent->__call('runEnd', Array)
#13 C:\AppServ\www\epretest\index.php(11): CWebApplication->runEnd('web')
#14 {main}
REQUEST_URI=/epretest/index.php?r=exam/save
HTTP_REFERER=http://localhost/epretest/index.php?r=exam&id=320b$7$3$5t97%3C22403
---
2014/04/29 20:38:21 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'The requested page does not exist.' in C:\AppServ\www\epretest\protected\controllers\web\ExamController.php:215
Stack trace:
#0 C:\AppServ\www\epretest\protected\controllers\web\ExamController.php(233): ExamController->loadTestRecord(NULL)
#1 C:\AppServ\www\epretest\protected\controllers\web\ExamController.php(128): ExamController->saveExamForm()
#2 C:\AppServ\www\epretest\framework\web\actions\CInlineAction.php(50): ExamController->actionSubmit()
#3 C:\AppServ\www\epretest\framework\web\CController.php(309): CInlineAction->runWithParams(Array)
#4 C:\AppServ\www\epretest\framework\web\CController.php(287): CController->runAction(Object(CInlineAction))
#5 C:\AppServ\www\epretest\framework\web\CController.php(266): CController->runActionWithFilters(Object(CInlineAction), Array)
#6 C:\AppServ\www\epretest\framework\web\CWebApplication.php(276): CController->run('submit')
#7 C:\AppServ\www\epretest\framework\web\CWebApplication.php(135): CWebApplication->runController('exam/submit')
#8 C:\AppServ\www\epretest\framework\base\CApplication.php(162): CWebApplication->processRequest()
#9 C:\AppServ\www\epretest\protected\components\WebApplicationEndBehavior.php(26): CApplication->run()
#10 [internal function]: WebApplicationEndBehavior->runEnd('web')
#11 C:\AppServ\www\epretest\framework\base\CComponent.php(261): call_user_func_array(Array, Array)
#12 [internal function]: CComponent->__call('runEnd', Array)
#13 C:\AppServ\www\epretest\index.php(11): CWebApplication->runEnd('web')
#14 {main}
REQUEST_URI=/epretest/index.php?r=exam/submit&id=7
HTTP_REFERER=http://localhost/epretest/index.php?r=exam&id=320b$7$3$5t97%3C22403
---
2014/04/29 20:48:40 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'The requested page does not exist.' in C:\AppServ\www\epretest\protected\controllers\web\ExamController.php:215
Stack trace:
#0 C:\AppServ\www\epretest\protected\controllers\web\ExamController.php(233): ExamController->loadTestRecord(NULL)
#1 C:\AppServ\www\epretest\protected\controllers\web\ExamController.php(128): ExamController->saveExamForm()
#2 C:\AppServ\www\epretest\framework\web\actions\CInlineAction.php(50): ExamController->actionSubmit()
#3 C:\AppServ\www\epretest\framework\web\CController.php(309): CInlineAction->runWithParams(Array)
#4 C:\AppServ\www\epretest\framework\web\CController.php(287): CController->runAction(Object(CInlineAction))
#5 C:\AppServ\www\epretest\framework\web\CController.php(266): CController->runActionWithFilters(Object(CInlineAction), Array)
#6 C:\AppServ\www\epretest\framework\web\CWebApplication.php(276): CController->run('submit')
#7 C:\AppServ\www\epretest\framework\web\CWebApplication.php(135): CWebApplication->runController('exam/submit')
#8 C:\AppServ\www\epretest\framework\base\CApplication.php(162): CWebApplication->processRequest()
#9 C:\AppServ\www\epretest\protected\components\WebApplicationEndBehavior.php(26): CApplication->run()
#10 [internal function]: WebApplicationEndBehavior->runEnd('web')
#11 C:\AppServ\www\epretest\framework\base\CComponent.php(261): call_user_func_array(Array, Array)
#12 [internal function]: CComponent->__call('runEnd', Array)
#13 C:\AppServ\www\epretest\index.php(11): CWebApplication->runEnd('web')
#14 {main}
REQUEST_URI=/epretest/index.php?r=exam/submit
HTTP_REFERER=http://localhost/epretest/index.php?r=exam&id=320b$7$3$5t97%3C10172
---
2014/04/30 11:05:37 [warning] [application] Failed to set unsafe attribute "hddnValue" of "LoginForm".
in C:\AppServ\www\epretest\protected\controllers\web\SiteController.php (265)
in C:\AppServ\www\epretest\protected\components\WebApplicationEndBehavior.php (26)
in C:\AppServ\www\epretest\index.php (11)
2014/04/30 15:00:17 [warning] [application] Failed to set unsafe attribute "hddnValue" of "LoginForm".
in C:\AppServ\www\epretest\protected\controllers\web\SiteController.php (265)
in C:\AppServ\www\epretest\protected\components\WebApplicationEndBehavior.php (26)
in C:\AppServ\www\epretest\index.php (11)
2014/04/30 15:41:46 [warning] [application] Failed to set unsafe attribute "hddnValue" of "LoginForm".
in C:\AppServ\www\epretest\protected\controllers\web\SiteController.php (265)
in C:\AppServ\www\epretest\protected\components\WebApplicationEndBehavior.php (26)
in C:\AppServ\www\epretest\index.php (11)
2014/05/01 11:07:24 [warning] [application] Failed to set unsafe attribute "hddnValue" of "LoginForm".
in C:\AppServ\www\epretest\protected\controllers\web\SiteController.php (265)
in C:\AppServ\www\epretest\protected\components\WebApplicationEndBehavior.php (26)
in C:\AppServ\www\epretest\index.php (11)
2014/05/01 16:17:39 [warning] [application] Failed to set unsafe attribute "hddnValue" of "LoginForm".
in C:\AppServ\www\epretest\protected\controllers\web\SiteController.php (265)
in C:\AppServ\www\epretest\protected\components\WebApplicationEndBehavior.php (26)
in C:\AppServ\www\epretest\index.php (11)
2014/05/01 22:09:13 [error] [php] Cannot modify header information - headers already sent by (output started at C:\AppServ\www\epretest\protected\controllers\web\ExamController.php:115) (C:\AppServ\www\epretest\framework\web\CHttpRequest.php:763)
Stack trace:
#0 C:\AppServ\www\epretest\protected\controllers\web\ExamController.php(116): ExamController->redirect()
#1 C:\AppServ\www\epretest\framework\web\actions\CInlineAction.php(50): ExamController->actionSubmit()
#2 C:\AppServ\www\epretest\framework\web\CController.php(309): CInlineAction->runWithParams()
#3 C:\AppServ\www\epretest\framework\web\CController.php(287): ExamController->runAction()
#4 C:\AppServ\www\epretest\framework\web\CController.php(266): ExamController->runActionWithFilters()
#5 C:\AppServ\www\epretest\framework\web\CWebApplication.php(276): ExamController->run()
#6 C:\AppServ\www\epretest\framework\web\CWebApplication.php(135): CWebApplication->runController()
#7 C:\AppServ\www\epretest\framework\base\CApplication.php(162): CWebApplication->processRequest()
#8 C:\AppServ\www\epretest\protected\components\WebApplicationEndBehavior.php(26): CWebApplication->run()
#9 unknown(0): WebApplicationEndBehavior->runEnd()
#10 C:\AppServ\www\epretest\framework\base\CComponent.php(261): call_user_func_array()
#11 unknown(0): CWebApplication->__call()
#12 C:\AppServ\www\epretest\index.php(11): CWebApplication->runEnd()
REQUEST_URI=/epretest/index.php?r=exam/submit&id=27
in C:\AppServ\www\epretest\protected\controllers\web\ExamController.php (116)
in C:\AppServ\www\epretest\protected\components\WebApplicationEndBehavior.php (26)
in C:\AppServ\www\epretest\index.php (11)
2014/05/01 22:09:22 [error] [php] Cannot modify header information - headers already sent by (output started at C:\AppServ\www\epretest\protected\controllers\web\ExamController.php:115) (C:\AppServ\www\epretest\framework\web\CHttpRequest.php:763)
Stack trace:
#0 C:\AppServ\www\epretest\protected\controllers\web\ExamController.php(116): ExamController->redirect()
#1 C:\AppServ\www\epretest\framework\web\actions\CInlineAction.php(50): ExamController->actionSubmit()
#2 C:\AppServ\www\epretest\framework\web\CController.php(309): CInlineAction->runWithParams()
#3 C:\AppServ\www\epretest\framework\web\CController.php(287): ExamController->runAction()
#4 C:\AppServ\www\epretest\framework\web\CController.php(266): ExamController->runActionWithFilters()
#5 C:\AppServ\www\epretest\framework\web\CWebApplication.php(276): ExamController->run()
#6 C:\AppServ\www\epretest\framework\web\CWebApplication.php(135): CWebApplication->runController()
#7 C:\AppServ\www\epretest\framework\base\CApplication.php(162): CWebApplication->processRequest()
#8 C:\AppServ\www\epretest\protected\components\WebApplicationEndBehavior.php(26): CWebApplication->run()
#9 unknown(0): WebApplicationEndBehavior->runEnd()
#10 C:\AppServ\www\epretest\framework\base\CComponent.php(261): call_user_func_array()
#11 unknown(0): CWebApplication->__call()
#12 C:\AppServ\www\epretest\index.php(11): CWebApplication->runEnd()
REQUEST_URI=/epretest/index.php?r=exam/submit&id=1
in C:\AppServ\www\epretest\protected\controllers\web\ExamController.php (116)
in C:\AppServ\www\epretest\protected\components\WebApplicationEndBehavior.php (26)
in C:\AppServ\www\epretest\index.php (11)
2014/05/01 22:09:33 [error] [php] Cannot modify header information - headers already sent by (output started at C:\AppServ\www\epretest\protected\controllers\web\ExamController.php:115) (C:\AppServ\www\epretest\framework\web\CHttpRequest.php:763)
Stack trace:
#0 C:\AppServ\www\epretest\protected\controllers\web\ExamController.php(116): ExamController->redirect()
#1 C:\AppServ\www\epretest\framework\web\actions\CInlineAction.php(50): ExamController->actionSubmit()
#2 C:\AppServ\www\epretest\framework\web\CController.php(309): CInlineAction->runWithParams()
#3 C:\AppServ\www\epretest\framework\web\CController.php(287): ExamController->runAction()
#4 C:\AppServ\www\epretest\framework\web\CController.php(266): ExamController->runActionWithFilters()
#5 C:\AppServ\www\epretest\framework\web\CWebApplication.php(276): ExamController->run()
#6 C:\AppServ\www\epretest\framework\web\CWebApplication.php(135): CWebApplication->runController()
#7 C:\AppServ\www\epretest\framework\base\CApplication.php(162): CWebApplication->processRequest()
#8 C:\AppServ\www\epretest\protected\components\WebApplicationEndBehavior.php(26): CWebApplication->run()
#9 unknown(0): WebApplicationEndBehavior->runEnd()
#10 C:\AppServ\www\epretest\framework\base\CComponent.php(261): call_user_func_array()
#11 unknown(0): CWebApplication->__call()
#12 C:\AppServ\www\epretest\index.php(11): CWebApplication->runEnd()
REQUEST_URI=/epretest/index.php?r=exam/submit&id=27
in C:\AppServ\www\epretest\protected\controllers\web\ExamController.php (116)
in C:\AppServ\www\epretest\protected\components\WebApplicationEndBehavior.php (26)
in C:\AppServ\www\epretest\index.php (11)
2014/05/01 22:10:59 [error] [php] Cannot modify header information - headers already sent by (output started at C:\AppServ\www\epretest\protected\controllers\web\ExamController.php:115) (C:\AppServ\www\epretest\framework\web\CHttpRequest.php:763)
Stack trace:
#0 C:\AppServ\www\epretest\protected\controllers\web\ExamController.php(116): ExamController->redirect()
#1 C:\AppServ\www\epretest\framework\web\actions\CInlineAction.php(50): ExamController->actionSubmit()
#2 C:\AppServ\www\epretest\framework\web\CController.php(309): CInlineAction->runWithParams()
#3 C:\AppServ\www\epretest\framework\web\CController.php(287): ExamController->runAction()
#4 C:\AppServ\www\epretest\framework\web\CController.php(266): ExamController->runActionWithFilters()
#5 C:\AppServ\www\epretest\framework\web\CWebApplication.php(276): ExamController->run()
#6 C:\AppServ\www\epretest\framework\web\CWebApplication.php(135): CWebApplication->runController()
#7 C:\AppServ\www\epretest\framework\base\CApplication.php(162): CWebApplication->processRequest()
#8 C:\AppServ\www\epretest\protected\components\WebApplicationEndBehavior.php(26): CWebApplication->run()
#9 unknown(0): WebApplicationEndBehavior->runEnd()
#10 C:\AppServ\www\epretest\framework\base\CComponent.php(261): call_user_func_array()
#11 unknown(0): CWebApplication->__call()
#12 C:\AppServ\www\epretest\index.php(11): CWebApplication->runEnd()
REQUEST_URI=/epretest/index.php?r=exam/submit&id=21
in C:\AppServ\www\epretest\protected\controllers\web\ExamController.php (116)
in C:\AppServ\www\epretest\protected\components\WebApplicationEndBehavior.php (26)
in C:\AppServ\www\epretest\index.php (11)
2014/05/01 23:24:03 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'The system is unable to find the requested action "submit?id=2342$29".' in C:\AppServ\www\epretest\framework\web\CController.php:484
Stack trace:
#0 C:\AppServ\www\epretest\framework\web\CController.php(271): CController->missingAction('submit?id=2342$...')
#1 C:\AppServ\www\epretest\framework\web\CWebApplication.php(276): CController->run('submit?id=2342$...')
#2 C:\AppServ\www\epretest\framework\web\CWebApplication.php(135): CWebApplication->runController('exam/submit?id=...')
#3 C:\AppServ\www\epretest\framework\base\CApplication.php(162): CWebApplication->processRequest()
#4 C:\AppServ\www\epretest\protected\components\WebApplicationEndBehavior.php(26): CApplication->run()
#5 [internal function]: WebApplicationEndBehavior->runEnd('web')
#6 C:\AppServ\www\epretest\framework\base\CComponent.php(261): call_user_func_array(Array, Array)
#7 [internal function]: CComponent->__call('runEnd', Array)
#8 C:\AppServ\www\epretest\index.php(11): CWebApplication->runEnd('web')
#9 {main}
REQUEST_URI=/epretest/index.php?r=exam/submit?id=2342$29
HTTP_REFERER=http://localhost/epretest/index.php?r=exam&id=32@%3$29$aw8
---
2014/05/01 23:26:07 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'The system is unable to find the requested action "submit?id=29".' in C:\AppServ\www\epretest\framework\web\CController.php:484
Stack trace:
#0 C:\AppServ\www\epretest\framework\web\CController.php(271): CController->missingAction('submit?id=29')
#1 C:\AppServ\www\epretest\framework\web\CWebApplication.php(276): CController->run('submit?id=29')
#2 C:\AppServ\www\epretest\framework\web\CWebApplication.php(135): CWebApplication->runController('exam/submit?id=...')
#3 C:\AppServ\www\epretest\framework\base\CApplication.php(162): CWebApplication->processRequest()
#4 C:\AppServ\www\epretest\protected\components\WebApplicationEndBehavior.php(26): CApplication->run()
#5 [internal function]: WebApplicationEndBehavior->runEnd('web')
#6 C:\AppServ\www\epretest\framework\base\CComponent.php(261): call_user_func_array(Array, Array)
#7 [internal function]: CComponent->__call('runEnd', Array)
#8 C:\AppServ\www\epretest\index.php(11): CWebApplication->runEnd('web')
#9 {main}
REQUEST_URI=/epretest/index.php?r=exam/submit?id=29
HTTP_REFERER=http://localhost/epretest/index.php?r=exam&id=32@%3$29$aw8
---
2014/05/01 23:27:38 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'The system is unable to find the requested action "submit?id=29".' in C:\AppServ\www\epretest\framework\web\CController.php:484
Stack trace:
#0 C:\AppServ\www\epretest\framework\web\CController.php(271): CController->missingAction('submit?id=29')
#1 C:\AppServ\www\epretest\framework\web\CWebApplication.php(276): CController->run('submit?id=29')
#2 C:\AppServ\www\epretest\framework\web\CWebApplication.php(135): CWebApplication->runController('exam/submit?id=...')
#3 C:\AppServ\www\epretest\framework\base\CApplication.php(162): CWebApplication->processRequest()
#4 C:\AppServ\www\epretest\protected\components\WebApplicationEndBehavior.php(26): CApplication->run()
#5 [internal function]: WebApplicationEndBehavior->runEnd('web')
#6 C:\AppServ\www\epretest\framework\base\CComponent.php(261): call_user_func_array(Array, Array)
#7 [internal function]: CComponent->__call('runEnd', Array)
#8 C:\AppServ\www\epretest\index.php(11): CWebApplication->runEnd('web')
#9 {main}
REQUEST_URI=/epretest/index.php?r=exam/submit?id=29
HTTP_REFERER=http://localhost/epretest/index.php?r=exam/index&id=%2302B13%2429
---
2014/05/01 23:29:42 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'The system is unable to find the requested action "submit?id=29".' in C:\AppServ\www\epretest\framework\web\CController.php:484
Stack trace:
#0 C:\AppServ\www\epretest\framework\web\CController.php(271): CController->missingAction('submit?id=29')
#1 C:\AppServ\www\epretest\framework\web\CWebApplication.php(276): CController->run('submit?id=29')
#2 C:\AppServ\www\epretest\framework\web\CWebApplication.php(135): CWebApplication->runController('exam/submit?id=...')
#3 C:\AppServ\www\epretest\framework\base\CApplication.php(162): CWebApplication->processRequest()
#4 C:\AppServ\www\epretest\protected\components\WebApplicationEndBehavior.php(26): CApplication->run()
#5 [internal function]: WebApplicationEndBehavior->runEnd('web')
#6 C:\AppServ\www\epretest\framework\base\CComponent.php(261): call_user_func_array(Array, Array)
#7 [internal function]: CComponent->__call('runEnd', Array)
#8 C:\AppServ\www\epretest\index.php(11): CWebApplication->runEnd('web')
#9 {main}
REQUEST_URI=/epretest/index.php?r=exam/submit?id=29
HTTP_REFERER=http://localhost/epretest/index.php?r=exam&id=32@%3$29$aw8
---
2014/05/01 23:30:06 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'The system is unable to find the requested action "submit?id=29".' in C:\AppServ\www\epretest\framework\web\CController.php:484
Stack trace:
#0 C:\AppServ\www\epretest\framework\web\CController.php(271): CController->missingAction('submit?id=29')
#1 C:\AppServ\www\epretest\framework\web\CWebApplication.php(276): CController->run('submit?id=29')
#2 C:\AppServ\www\epretest\framework\web\CWebApplication.php(135): CWebApplication->runController('exam/submit?id=...')
#3 C:\AppServ\www\epretest\framework\base\CApplication.php(162): CWebApplication->processRequest()
#4 C:\AppServ\www\epretest\protected\components\WebApplicationEndBehavior.php(26): CApplication->run()
#5 [internal function]: WebApplicationEndBehavior->runEnd('web')
#6 C:\AppServ\www\epretest\framework\base\CComponent.php(261): call_user_func_array(Array, Array)
#7 [internal function]: CComponent->__call('runEnd', Array)
#8 C:\AppServ\www\epretest\index.php(11): CWebApplication->runEnd('web')
#9 {main}
REQUEST_URI=/epretest/index.php?r=exam/submit?id=29
HTTP_REFERER=http://localhost/epretest/index.php?r=exam&id=32@%3$29$aw8
---
2014/05/01 23:32:26 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'The system is unable to find the requested action "submit?id=30".' in C:\AppServ\www\epretest\framework\web\CController.php:484
Stack trace:
#0 C:\AppServ\www\epretest\framework\web\CController.php(271): CController->missingAction('submit?id=30')
#1 C:\AppServ\www\epretest\framework\web\CWebApplication.php(276): CController->run('submit?id=30')
#2 C:\AppServ\www\epretest\framework\web\CWebApplication.php(135): CWebApplication->runController('exam/submit?id=...')
#3 C:\AppServ\www\epretest\framework\base\CApplication.php(162): CWebApplication->processRequest()
#4 C:\AppServ\www\epretest\protected\components\WebApplicationEndBehavior.php(26): CApplication->run()
#5 [internal function]: WebApplicationEndBehavior->runEnd('web')
#6 C:\AppServ\www\epretest\framework\base\CComponent.php(261): call_user_func_array(Array, Array)
#7 [internal function]: CComponent->__call('runEnd', Array)
#8 C:\AppServ\www\epretest\index.php(11): CWebApplication->runEnd('web')
#9 {main}
REQUEST_URI=/epretest/index.php?r=exam/submit?id=30
HTTP_REFERER=http://localhost/epretest/index.php?r=exam/index&id=%2302B13%2430
---
2014/05/01 23:33:03 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'The system is unable to find the requested action "submit?id=21".' in C:\AppServ\www\epretest\framework\web\CController.php:484
Stack trace:
#0 C:\AppServ\www\epretest\framework\web\CController.php(271): CController->missingAction('submit?id=21')
#1 C:\AppServ\www\epretest\framework\web\CWebApplication.php(276): CController->run('submit?id=21')
#2 C:\AppServ\www\epretest\framework\web\CWebApplication.php(135): CWebApplication->runController('exam/submit?id=...')
#3 C:\AppServ\www\epretest\framework\base\CApplication.php(162): CWebApplication->processRequest()
#4 C:\AppServ\www\epretest\protected\components\WebApplicationEndBehavior.php(26): CApplication->run()
#5 [internal function]: WebApplicationEndBehavior->runEnd('web')
#6 C:\AppServ\www\epretest\framework\base\CComponent.php(261): call_user_func_array(Array, Array)
#7 [internal function]: CComponent->__call('runEnd', Array)
#8 C:\AppServ\www\epretest\index.php(11): CWebApplication->runEnd('web')
#9 {main}
REQUEST_URI=/epretest/index.php?r=exam/submit?id=21
HTTP_REFERER=http://localhost/epretest/index.php?r=exam/index&id=%2302B13%2421
---
2014/05/01 23:36:28 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'The system is unable to find the requested action "submit?id=21".' in C:\AppServ\www\epretest\framework\web\CController.php:484
Stack trace:
#0 C:\AppServ\www\epretest\framework\web\CController.php(271): CController->missingAction('submit?id=21')
#1 C:\AppServ\www\epretest\framework\web\CWebApplication.php(276): CController->run('submit?id=21')
#2 C:\AppServ\www\epretest\framework\web\CWebApplication.php(135): CWebApplication->runController('exam/submit?id=...')
#3 C:\AppServ\www\epretest\framework\base\CApplication.php(162): CWebApplication->processRequest()
#4 C:\AppServ\www\epretest\protected\components\WebApplicationEndBehavior.php(26): CApplication->run()
#5 [internal function]: WebApplicationEndBehavior->runEnd('web')
#6 C:\AppServ\www\epretest\framework\base\CComponent.php(261): call_user_func_array(Array, Array)
#7 [internal function]: CComponent->__call('runEnd', Array)
#8 C:\AppServ\www\epretest\index.php(11): CWebApplication->runEnd('web')
#9 {main}
REQUEST_URI=/epretest/index.php?r=exam/submit?id=21
HTTP_REFERER=http://localhost/epretest/index.php?r=exam/index&id=%2302B13%2421
---
2014/05/01 23:37:15 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'The system is unable to find the requested action "submit?id=21".' in C:\AppServ\www\epretest\framework\web\CController.php:484
Stack trace:
#0 C:\AppServ\www\epretest\framework\web\CController.php(271): CController->missingAction('submit?id=21')
#1 C:\AppServ\www\epretest\framework\web\CWebApplication.php(276): CController->run('submit?id=21')
#2 C:\AppServ\www\epretest\framework\web\CWebApplication.php(135): CWebApplication->runController('exam/submit?id=...')
#3 C:\AppServ\www\epretest\framework\base\CApplication.php(162): CWebApplication->processRequest()
#4 C:\AppServ\www\epretest\protected\components\WebApplicationEndBehavior.php(26): CApplication->run()
#5 [internal function]: WebApplicationEndBehavior->runEnd('web')
#6 C:\AppServ\www\epretest\framework\base\CComponent.php(261): call_user_func_array(Array, Array)
#7 [internal function]: CComponent->__call('runEnd', Array)
#8 C:\AppServ\www\epretest\index.php(11): CWebApplication->runEnd('web')
#9 {main}
REQUEST_URI=/epretest/index.php?r=exam/submit?id=21
---
2014/05/01 23:37:54 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'The system is unable to find the requested action "submit?id=21".' in C:\AppServ\www\epretest\framework\web\CController.php:484
Stack trace:
#0 C:\AppServ\www\epretest\framework\web\CController.php(271): CController->missingAction('submit?id=21')
#1 C:\AppServ\www\epretest\framework\web\CWebApplication.php(276): CController->run('submit?id=21')
#2 C:\AppServ\www\epretest\framework\web\CWebApplication.php(135): CWebApplication->runController('exam/submit?id=...')
#3 C:\AppServ\www\epretest\framework\base\CApplication.php(162): CWebApplication->processRequest()
#4 C:\AppServ\www\epretest\protected\components\WebApplicationEndBehavior.php(26): CApplication->run()
#5 [internal function]: WebApplicationEndBehavior->runEnd('web')
#6 C:\AppServ\www\epretest\framework\base\CComponent.php(261): call_user_func_array(Array, Array)
#7 [internal function]: CComponent->__call('runEnd', Array)
#8 C:\AppServ\www\epretest\index.php(11): CWebApplication->runEnd('web')
#9 {main}
REQUEST_URI=/epretest/index.php?r=exam/submit?id=21
---
2014/05/01 23:39:45 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'The system is unable to find the requested action "submit?id=21".' in C:\AppServ\www\epretest\framework\web\CController.php:484
Stack trace:
#0 C:\AppServ\www\epretest\framework\web\CController.php(271): CController->missingAction('submit?id=21')
#1 C:\AppServ\www\epretest\framework\web\CWebApplication.php(276): CController->run('submit?id=21')
#2 C:\AppServ\www\epretest\framework\web\CWebApplication.php(135): CWebApplication->runController('exam/submit?id=...')
#3 C:\AppServ\www\epretest\framework\base\CApplication.php(162): CWebApplication->processRequest()
#4 C:\AppServ\www\epretest\protected\components\WebApplicationEndBehavior.php(26): CApplication->run()
#5 [internal function]: WebApplicationEndBehavior->runEnd('web')
#6 C:\AppServ\www\epretest\framework\base\CComponent.php(261): call_user_func_array(Array, Array)
#7 [internal function]: CComponent->__call('runEnd', Array)
#8 C:\AppServ\www\epretest\index.php(11): CWebApplication->runEnd('web')
#9 {main}
REQUEST_URI=/epretest/index.php?r=exam/submit?id=21
HTTP_REFERER=http://localhost/epretest/index.php?r=exam/index&id=%2302B13%2421
---
2014/05/01 23:42:44 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'The system is unable to find the requested action "submit?id=21".' in C:\AppServ\www\epretest\framework\web\CController.php:484
Stack trace:
#0 C:\AppServ\www\epretest\framework\web\CController.php(271): CController->missingAction('submit?id=21')
#1 C:\AppServ\www\epretest\framework\web\CWebApplication.php(276): CController->run('submit?id=21')
#2 C:\AppServ\www\epretest\framework\web\CWebApplication.php(135): CWebApplication->runController('exam/submit?id=...')
#3 C:\AppServ\www\epretest\framework\base\CApplication.php(162): CWebApplication->processRequest()
#4 C:\AppServ\www\epretest\protected\components\WebApplicationEndBehavior.php(26): CApplication->run()
#5 [internal function]: WebApplicationEndBehavior->runEnd('web')
#6 C:\AppServ\www\epretest\framework\base\CComponent.php(261): call_user_func_array(Array, Array)
#7 [internal function]: CComponent->__call('runEnd', Array)
#8 C:\AppServ\www\epretest\index.php(11): CWebApplication->runEnd('web')
#9 {main}
REQUEST_URI=/epretest/index.php?r=exam/submit?id=21
HTTP_REFERER=http://localhost/epretest/index.php?r=exam/index&id=%2302B13%2421
---
2014/05/01 23:49:18 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'The system is unable to find the requested action "submit?sid=21".' in C:\AppServ\www\epretest\framework\web\CController.php:484
Stack trace:
#0 C:\AppServ\www\epretest\framework\web\CController.php(271): CController->missingAction('submit?sid=21')
#1 C:\AppServ\www\epretest\framework\web\CWebApplication.php(276): CController->run('submit?sid=21')
#2 C:\AppServ\www\epretest\framework\web\CWebApplication.php(135): CWebApplication->runController('exam/submit?sid...')
#3 C:\AppServ\www\epretest\framework\base\CApplication.php(162): CWebApplication->processRequest()
#4 C:\AppServ\www\epretest\protected\components\WebApplicationEndBehavior.php(26): CApplication->run()
#5 [internal function]: WebApplicationEndBehavior->runEnd('web')
#6 C:\AppServ\www\epretest\framework\base\CComponent.php(261): call_user_func_array(Array, Array)
#7 [internal function]: CComponent->__call('runEnd', Array)
#8 C:\AppServ\www\epretest\index.php(11): CWebApplication->runEnd('web')
#9 {main}
REQUEST_URI=/epretest/index.php?r=exam/submit?sid=21
HTTP_REFERER=http://localhost/epretest/index.php?r=exam/index&id=%2302B13%2421
---
2014/05/01 23:53:49 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'The system is unable to find the requested action "submit?id=21".' in C:\AppServ\www\epretest\framework\web\CController.php:484
Stack trace:
#0 C:\AppServ\www\epretest\framework\web\CController.php(271): CController->missingAction('submit?id=21')
#1 C:\AppServ\www\epretest\framework\web\CWebApplication.php(276): CController->run('submit?id=21')
#2 C:\AppServ\www\epretest\framework\web\CWebApplication.php(135): CWebApplication->runController('exam/submit?id=...')
#3 C:\AppServ\www\epretest\framework\base\CApplication.php(162): CWebApplication->processRequest()
#4 C:\AppServ\www\epretest\protected\components\WebApplicationEndBehavior.php(26): CApplication->run()
#5 [internal function]: WebApplicationEndBehavior->runEnd('web')
#6 C:\AppServ\www\epretest\framework\base\CComponent.php(261): call_user_func_array(Array, Array)
#7 [internal function]: CComponent->__call('runEnd', Array)
#8 C:\AppServ\www\epretest\index.php(11): CWebApplication->runEnd('web')
#9 {main}
REQUEST_URI=/epretest/index.php?r=exam/submit?id=21
HTTP_REFERER=http://localhost/epretest/index.php?r=exam&id=32@%3$21$aw8
---
2014/05/01 23:54:02 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'The system is unable to find the requested action "submit?id=21".' in C:\AppServ\www\epretest\framework\web\CController.php:484
Stack trace:
#0 C:\AppServ\www\epretest\framework\web\CController.php(271): CController->missingAction('submit?id=21')
#1 C:\AppServ\www\epretest\framework\web\CWebApplication.php(276): CController->run('submit?id=21')
#2 C:\AppServ\www\epretest\framework\web\CWebApplication.php(135): CWebApplication->runController('exam/submit?id=...')
#3 C:\AppServ\www\epretest\framework\base\CApplication.php(162): CWebApplication->processRequest()
#4 C:\AppServ\www\epretest\protected\components\WebApplicationEndBehavior.php(26): CApplication->run()
#5 [internal function]: WebApplicationEndBehavior->runEnd('web')
#6 C:\AppServ\www\epretest\framework\base\CComponent.php(261): call_user_func_array(Array, Array)
#7 [internal function]: CComponent->__call('runEnd', Array)
#8 C:\AppServ\www\epretest\index.php(11): CWebApplication->runEnd('web')
#9 {main}
REQUEST_URI=/epretest/index.php?r=exam/submit?id=21
HTTP_REFERER=http://localhost/epretest/index.php?r=exam/index&id=%2302B13%2421
---
2014/05/01 23:54:49 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'The system is unable to find the requested action "submit?id=21".' in C:\AppServ\www\epretest\framework\web\CController.php:484
Stack trace:
#0 C:\AppServ\www\epretest\framework\web\CController.php(271): CController->missingAction('submit?id=21')
#1 C:\AppServ\www\epretest\framework\web\CWebApplication.php(276): CController->run('submit?id=21')
#2 C:\AppServ\www\epretest\framework\web\CWebApplication.php(135): CWebApplication->runController('exam/submit?id=...')
#3 C:\AppServ\www\epretest\framework\base\CApplication.php(162): CWebApplication->processRequest()
#4 C:\AppServ\www\epretest\protected\components\WebApplicationEndBehavior.php(26): CApplication->run()
#5 [internal function]: WebApplicationEndBehavior->runEnd('web')
#6 C:\AppServ\www\epretest\framework\base\CComponent.php(261): call_user_func_array(Array, Array)
#7 [internal function]: CComponent->__call('runEnd', Array)
#8 C:\AppServ\www\epretest\index.php(11): CWebApplication->runEnd('web')
#9 {main}
REQUEST_URI=/epretest/index.php?r=exam/submit?id=21
HTTP_REFERER=http://localhost/epretest/index.php?r=exam/index&id=%2302B13%2421
---
2014/05/01 23:57:44 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'The system is unable to find the requested action "submit?id=21".' in C:\AppServ\www\epretest\framework\web\CController.php:484
Stack trace:
#0 C:\AppServ\www\epretest\framework\web\CController.php(271): CController->missingAction('submit?id=21')
#1 C:\AppServ\www\epretest\framework\web\CWebApplication.php(276): CController->run('submit?id=21')
#2 C:\AppServ\www\epretest\framework\web\CWebApplication.php(135): CWebApplication->runController('exam/submit?id=...')
#3 C:\AppServ\www\epretest\framework\base\CApplication.php(162): CWebApplication->processRequest()
#4 C:\AppServ\www\epretest\protected\components\WebApplicationEndBehavior.php(26): CApplication->run()
#5 [internal function]: WebApplicationEndBehavior->runEnd('web')
#6 C:\AppServ\www\epretest\framework\base\CComponent.php(261): call_user_func_array(Array, Array)
#7 [internal function]: CComponent->__call('runEnd', Array)
#8 C:\AppServ\www\epretest\index.php(11): CWebApplication->runEnd('web')
#9 {main}
REQUEST_URI=/epretest/index.php?r=exam/submit?id=21
HTTP_REFERER=http://localhost/epretest/index.php?r=exam/index&id=%2302B13%2421
---
2014/05/02 00:00:32 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'The system is unable to find the requested action "submit?".' in C:\AppServ\www\epretest\framework\web\CController.php:484
Stack trace:
#0 C:\AppServ\www\epretest\framework\web\CController.php(271): CController->missingAction('submit?')
#1 C:\AppServ\www\epretest\framework\web\CWebApplication.php(276): CController->run('submit?')
#2 C:\AppServ\www\epretest\framework\web\CWebApplication.php(135): CWebApplication->runController('exam/submit?')
#3 C:\AppServ\www\epretest\framework\base\CApplication.php(162): CWebApplication->processRequest()
#4 C:\AppServ\www\epretest\protected\components\WebApplicationEndBehavior.php(26): CApplication->run()
#5 [internal function]: WebApplicationEndBehavior->runEnd('web')
#6 C:\AppServ\www\epretest\framework\base\CComponent.php(261): call_user_func_array(Array, Array)
#7 [internal function]: CComponent->__call('runEnd', Array)
#8 C:\AppServ\www\epretest\index.php(11): CWebApplication->runEnd('web')
#9 {main}
REQUEST_URI=/epretest/index.php?r=exam/submit?&id=21
---
2014/05/03 08:23:56 [warning] [application] Failed to set unsafe attribute "hddnValue" of "LoginForm".
in C:\AppServ\www\epretest\protected\controllers\web\SiteController.php (265)
in C:\AppServ\www\epretest\protected\components\WebApplicationEndBehavior.php (26)
in C:\AppServ\www\epretest\index.php (11)
2014/05/03 09:42:32 [error] [system.db.CDbCommand] CDbCommand::execute() failed: SQLSTATE[HY000]: General error: 1452 Cannot add or update a child row: a foreign key constraint fails (`epretest/esto_test_record`, CONSTRAINT `esto_test_record_ibfk_1` FOREIGN KEY (`exam_id`) REFERENCES `esto_exam` (`exam_id`)). The SQL statement executed was: INSERT INTO `esto_test_record` (`exam_id`, `student_id`, `score`, `date_attended`, `elapse_time`, `status`) VALUES (:exam_id, :student_id, :score, :date_attended, :elapse_time, :status).
in C:\AppServ\www\epretest\protected\models\TestRecord.php (224)
in C:\AppServ\www\epretest\protected\controllers\web\ExamController.php (192)
in C:\AppServ\www\epretest\protected\components\WebApplicationEndBehavior.php (26)
2014/05/03 09:42:32 [error] [exception.CDbException] exception 'CDbException' with message 'CDbCommand failed to execute the SQL statement: SQLSTATE[HY000]: General error: 1452 Cannot add or update a child row: a foreign key constraint fails (`epretest/esto_test_record`, CONSTRAINT `esto_test_record_ibfk_1` FOREIGN KEY (`exam_id`) REFERENCES `esto_exam` (`exam_id`)). The SQL statement executed was: INSERT INTO `esto_test_record` (`exam_id`, `student_id`, `score`, `date_attended`, `elapse_time`, `status`) VALUES (:exam_id, :student_id, :score, :date_attended, :elapse_time, :status)' in C:\AppServ\www\epretest\framework\db\CDbCommand.php:354
Stack trace:
#0 C:\AppServ\www\epretest\framework\db\CDbCommand.php(1178): CDbCommand->execute(Array)
#1 C:\AppServ\www\epretest\protected\models\TestRecord.php(224): CDbCommand->insert('esto_test_recor...', Array)
#2 C:\AppServ\www\epretest\protected\controllers\web\ExamController.php(192): TestRecord->saveFirstLogTestRecord('undefined', '8')
#3 C:\AppServ\www\epretest\framework\web\actions\CInlineAction.php(50): ExamController->actionUsecredit()
#4 C:\AppServ\www\epretest\framework\web\CController.php(309): CInlineAction->runWithParams(Array)
#5 C:\AppServ\www\epretest\framework\web\CController.php(287): CController->runAction(Object(CInlineAction))
#6 C:\AppServ\www\epretest\framework\web\CController.php(266): CController->runActionWithFilters(Object(CInlineAction), Array)
#7 C:\AppServ\www\epretest\framework\web\CWebApplication.php(276): CController->run('usecredit')
#8 C:\AppServ\www\epretest\framework\web\CWebApplication.php(135): CWebApplication->runController('exam/usecredit')
#9 C:\AppServ\www\epretest\framework\base\CApplication.php(162): CWebApplication->processRequest()
#10 C:\AppServ\www\epretest\protected\components\WebApplicationEndBehavior.php(26): CApplication->run()
#11 [internal function]: WebApplicationEndBehavior->runEnd('web')
#12 C:\AppServ\www\epretest\framework\base\CComponent.php(261): call_user_func_array(Array, Array)
#13 [internal function]: CComponent->__call('runEnd', Array)
#14 C:\AppServ\www\epretest\index.php(11): CWebApplication->runEnd('web')
#15 {main}
REQUEST_URI=/epretest/index.php?r=exam/usecredit&credit=undefined&id=undefined
HTTP_REFERER=http://localhost/epretest/index.php?r=exam&id=32@%3$30$aw8
---
2014/05/03 09:43:40 [error] [system.db.CDbCommand] CDbCommand::execute() failed: SQLSTATE[HY000]: General error: 1452 Cannot add or update a child row: a foreign key constraint fails (`epretest/esto_test_record`, CONSTRAINT `esto_test_record_ibfk_1` FOREIGN KEY (`exam_id`) REFERENCES `esto_exam` (`exam_id`)). The SQL statement executed was: INSERT INTO `esto_test_record` (`exam_id`, `student_id`, `score`, `date_attended`, `elapse_time`, `status`) VALUES (:exam_id, :student_id, :score, :date_attended, :elapse_time, :status).
in C:\AppServ\www\epretest\protected\models\TestRecord.php (224)
in C:\AppServ\www\epretest\protected\controllers\web\ExamController.php (192)
in C:\AppServ\www\epretest\protected\components\WebApplicationEndBehavior.php (26)
2014/05/03 09:43:40 [error] [exception.CDbException] exception 'CDbException' with message 'CDbCommand failed to execute the SQL statement: SQLSTATE[HY000]: General error: 1452 Cannot add or update a child row: a foreign key constraint fails (`epretest/esto_test_record`, CONSTRAINT `esto_test_record_ibfk_1` FOREIGN KEY (`exam_id`) REFERENCES `esto_exam` (`exam_id`)). The SQL statement executed was: INSERT INTO `esto_test_record` (`exam_id`, `student_id`, `score`, `date_attended`, `elapse_time`, `status`) VALUES (:exam_id, :student_id, :score, :date_attended, :elapse_time, :status)' in C:\AppServ\www\epretest\framework\db\CDbCommand.php:354
Stack trace:
#0 C:\AppServ\www\epretest\framework\db\CDbCommand.php(1178): CDbCommand->execute(Array)
#1 C:\AppServ\www\epretest\protected\models\TestRecord.php(224): CDbCommand->insert('esto_test_recor...', Array)
#2 C:\AppServ\www\epretest\protected\controllers\web\ExamController.php(192): TestRecord->saveFirstLogTestRecord('undefined', '8')
#3 C:\AppServ\www\epretest\framework\web\actions\CInlineAction.php(50): ExamController->actionUsecredit()
#4 C:\AppServ\www\epretest\framework\web\CController.php(309): CInlineAction->runWithParams(Array)
#5 C:\AppServ\www\epretest\framework\web\CController.php(287): CController->runAction(Object(CInlineAction))
#6 C:\AppServ\www\epretest\framework\web\CController.php(266): CController->runActionWithFilters(Object(CInlineAction), Array)
#7 C:\AppServ\www\epretest\framework\web\CWebApplication.php(276): CController->run('usecredit')
#8 C:\AppServ\www\epretest\framework\web\CWebApplication.php(135): CWebApplication->runController('exam/usecredit')
#9 C:\AppServ\www\epretest\framework\base\CApplication.php(162): CWebApplication->processRequest()
#10 C:\AppServ\www\epretest\protected\components\WebApplicationEndBehavior.php(26): CApplication->run()
#11 [internal function]: WebApplicationEndBehavior->runEnd('web')
#12 C:\AppServ\www\epretest\framework\base\CComponent.php(261): call_user_func_array(Array, Array)
#13 [internal function]: CComponent->__call('runEnd', Array)
#14 C:\AppServ\www\epretest\index.php(11): CWebApplication->runEnd('web')
#15 {main}
REQUEST_URI=/epretest/index.php?r=exam/usecredit&credit=undefined&id=undefined
HTTP_REFERER=http://localhost/epretest/index.php?r=exam&id=32@%3$24$aw8
---
2014/05/03 09:43:53 [error] [system.db.CDbCommand] CDbCommand::execute() failed: SQLSTATE[HY000]: General error: 1452 Cannot add or update a child row: a foreign key constraint fails (`epretest/esto_test_record`, CONSTRAINT `esto_test_record_ibfk_1` FOREIGN KEY (`exam_id`) REFERENCES `esto_exam` (`exam_id`)). The SQL statement executed was: INSERT INTO `esto_test_record` (`exam_id`, `student_id`, `score`, `date_attended`, `elapse_time`, `status`) VALUES (:exam_id, :student_id, :score, :date_attended, :elapse_time, :status).
in C:\AppServ\www\epretest\protected\models\TestRecord.php (224)
in C:\AppServ\www\epretest\protected\controllers\web\ExamController.php (192)
in C:\AppServ\www\epretest\protected\components\WebApplicationEndBehavior.php (26)
2014/05/03 09:43:53 [error] [exception.CDbException] exception 'CDbException' with message 'CDbCommand failed to execute the SQL statement: SQLSTATE[HY000]: General error: 1452 Cannot add or update a child row: a foreign key constraint fails (`epretest/esto_test_record`, CONSTRAINT `esto_test_record_ibfk_1` FOREIGN KEY (`exam_id`) REFERENCES `esto_exam` (`exam_id`)). The SQL statement executed was: INSERT INTO `esto_test_record` (`exam_id`, `student_id`, `score`, `date_attended`, `elapse_time`, `status`) VALUES (:exam_id, :student_id, :score, :date_attended, :elapse_time, :status)' in C:\AppServ\www\epretest\framework\db\CDbCommand.php:354
Stack trace:
#0 C:\AppServ\www\epretest\framework\db\CDbCommand.php(1178): CDbCommand->execute(Array)
#1 C:\AppServ\www\epretest\protected\models\TestRecord.php(224): CDbCommand->insert('esto_test_recor...', Array)
#2 C:\AppServ\www\epretest\protected\controllers\web\ExamController.php(192): TestRecord->saveFirstLogTestRecord('undefined', '8')
#3 C:\AppServ\www\epretest\framework\web\actions\CInlineAction.php(50): ExamController->actionUsecredit()
#4 C:\AppServ\www\epretest\framework\web\CController.php(309): CInlineAction->runWithParams(Array)
#5 C:\AppServ\www\epretest\framework\web\CController.php(287): CController->runAction(Object(CInlineAction))
#6 C:\AppServ\www\epretest\framework\web\CController.php(266): CController->runActionWithFilters(Object(CInlineAction), Array)
#7 C:\AppServ\www\epretest\framework\web\CWebApplication.php(276): CController->run('usecredit')
#8 C:\AppServ\www\epretest\framework\web\CWebApplication.php(135): CWebApplication->runController('exam/usecredit')
#9 C:\AppServ\www\epretest\framework\base\CApplication.php(162): CWebApplication->processRequest()
#10 C:\AppServ\www\epretest\protected\components\WebApplicationEndBehavior.php(26): CApplication->run()
#11 [internal function]: WebApplicationEndBehavior->runEnd('web')
#12 C:\AppServ\www\epretest\framework\base\CComponent.php(261): call_user_func_array(Array, Array)
#13 [internal function]: CComponent->__call('runEnd', Array)
#14 C:\AppServ\www\epretest\index.php(11): CWebApplication->runEnd('web')
#15 {main}
REQUEST_URI=/epretest/index.php?r=exam/usecredit&credit=undefined&id=undefined
HTTP_REFERER=http://localhost/epretest/index.php?r=exam&id=32@%3$24$aw8
---
2014/05/03 09:45:37 [error] [system.db.CDbCommand] CDbCommand::execute() failed: SQLSTATE[HY000]: General error: 1452 Cannot add or update a child row: a foreign key constraint fails (`epretest/esto_test_record`, CONSTRAINT `esto_test_record_ibfk_1` FOREIGN KEY (`exam_id`) REFERENCES `esto_exam` (`exam_id`)). The SQL statement executed was: INSERT INTO `esto_test_record` (`exam_id`, `student_id`, `score`, `date_attended`, `elapse_time`, `status`) VALUES (:exam_id, :student_id, :score, :date_attended, :elapse_time, :status).
in C:\AppServ\www\epretest\protected\models\TestRecord.php (224)
in C:\AppServ\www\epretest\protected\controllers\web\ExamController.php (192)
in C:\AppServ\www\epretest\protected\components\WebApplicationEndBehavior.php (26)
2014/05/03 09:45:37 [error] [exception.CDbException] exception 'CDbException' with message 'CDbCommand failed to execute the SQL statement: SQLSTATE[HY000]: General error: 1452 Cannot add or update a child row: a foreign key constraint fails (`epretest/esto_test_record`, CONSTRAINT `esto_test_record_ibfk_1` FOREIGN KEY (`exam_id`) REFERENCES `esto_exam` (`exam_id`)). The SQL statement executed was: INSERT INTO `esto_test_record` (`exam_id`, `student_id`, `score`, `date_attended`, `elapse_time`, `status`) VALUES (:exam_id, :student_id, :score, :date_attended, :elapse_time, :status)' in C:\AppServ\www\epretest\framework\db\CDbCommand.php:354
Stack trace:
#0 C:\AppServ\www\epretest\framework\db\CDbCommand.php(1178): CDbCommand->execute(Array)
#1 C:\AppServ\www\epretest\protected\models\TestRecord.php(224): CDbCommand->insert('esto_test_recor...', Array)
#2 C:\AppServ\www\epretest\protected\controllers\web\ExamController.php(192): TestRecord->saveFirstLogTestRecord('undefined', '8')
#3 C:\AppServ\www\epretest\framework\web\actions\CInlineAction.php(50): ExamController->actionUsecredit()
#4 C:\AppServ\www\epretest\framework\web\CController.php(309): CInlineAction->runWithParams(Array)
#5 C:\AppServ\www\epretest\framework\web\CController.php(287): CController->runAction(Object(CInlineAction))
#6 C:\AppServ\www\epretest\framework\web\CController.php(266): CController->runActionWithFilters(Object(CInlineAction), Array)
#7 C:\AppServ\www\epretest\framework\web\CWebApplication.php(276): CController->run('usecredit')
#8 C:\AppServ\www\epretest\framework\web\CWebApplication.php(135): CWebApplication->runController('exam/usecredit')
#9 C:\AppServ\www\epretest\framework\base\CApplication.php(162): CWebApplication->processRequest()
#10 C:\AppServ\www\epretest\protected\components\WebApplicationEndBehavior.php(26): CApplication->run()
#11 [internal function]: WebApplicationEndBehavior->runEnd('web')
#12 C:\AppServ\www\epretest\framework\base\CComponent.php(261): call_user_func_array(Array, Array)
#13 [internal function]: CComponent->__call('runEnd', Array)
#14 C:\AppServ\www\epretest\index.php(11): CWebApplication->runEnd('web')
#15 {main}
REQUEST_URI=/epretest/index.php?r=exam/usecredit&credit=undefined&id=undefined
HTTP_REFERER=http://localhost/epretest/index.php?r=exam&id=32@%3$24$aw8
---
2014/05/03 09:52:49 [error] [php] move_uploaded_file(C:\AppServ\www\epretest\protected/../uploads/pdf/uploaded_20140503095248.pdf) [<a href='function.move-uploaded-file'>function.move-uploaded-file</a>]: failed to open stream: No such file or directory (C:\AppServ\www\epretest\framework\web\CUploadedFile.php:197)
Stack trace:
#0 C:\AppServ\www\epretest\framework\web\actions\CInlineAction.php(50): ExamController->actionCreate()
#1 C:\AppServ\www\epretest\framework\web\CController.php(309): CInlineAction->runWithParams()
#2 C:\AppServ\www\epretest\framework\web\filters\CFilterChain.php(134): ExamController->runAction()
#3 C:\AppServ\www\epretest\framework\web\filters\CFilter.php(41): CFilterChain->run()
#4 C:\AppServ\www\epretest\framework\web\CController.php(1146): CAccessControlFilter->filter()
#5 C:\AppServ\www\epretest\framework\web\filters\CInlineFilter.php(59): ExamController->filterAccessControl()
#6 C:\AppServ\www\epretest\framework\web\filters\CFilterChain.php(131): CInlineFilter->filter()
#7 C:\AppServ\www\epretest\framework\web\CController.php(292): CFilterChain->run()
#8 C:\AppServ\www\epretest\framework\web\CController.php(266): ExamController->runActionWithFilters()
#9 C:\AppServ\www\epretest\framework\web\CWebApplication.php(276): ExamController->run()
#10 C:\AppServ\www\epretest\framework\web\CWebApplication.php(135): CWebApplication->runController()
#11 C:\AppServ\www\epretest\framework\base\CApplication.php(162): CWebApplication->processRequest()
#12 C:\AppServ\www\epretest\protected\components\WebApplicationEndBehavior.php(26): CWebApplication->run()
#13 unknown(0): WebApplicationEndBehavior->runEnd()
#14 C:\AppServ\www\epretest\framework\base\CComponent.php(261): call_user_func_array()
#15 unknown(0): CWebApplication->__call()
#16 C:\AppServ\www\epretest\admin.php(11): CWebApplication->runEnd()
REQUEST_URI=/epretest/admin/exam/create
in C:\AppServ\www\epretest\protected\controllers\admin\ExamController.php (88)
in C:\AppServ\www\epretest\protected\components\WebApplicationEndBehavior.php (26)
in C:\AppServ\www\epretest\admin.php (11)
2014/05/03 09:59:50 [error] [system.db.CDbCommand] CDbCommand::fetch() failed: SQLSTATE[HY093]: Invalid parameter number: parameter was not defined. The SQL statement executed was: SELECT COUNT(*) as total
FROM `esto_exam`
WHERE status=:status AND subject_id=:subject_id.
in C:\AppServ\www\epretest\protected\models\Exam.php (151)
in C:\AppServ\www\epretest\protected\views\web\student\view.php (204)
in C:\AppServ\www\epretest\protected\controllers\web\StudentController.php (144)
2014/05/03 09:59:50 [error] [exception.CDbException] exception 'CDbException' with message 'CDbCommand failed to execute the SQL statement: SQLSTATE[HY093]: Invalid parameter number: parameter was not defined. The SQL statement executed was: SELECT COUNT(*) as total
FROM `esto_exam`
WHERE status=:status AND subject_id=:subject_id' in C:\AppServ\www\epretest\framework\db\CDbCommand.php:528
Stack trace:
#0 C:\AppServ\www\epretest\framework\db\CDbCommand.php(408): CDbCommand->queryInternal('fetch', Array, Array)
#1 C:\AppServ\www\epretest\protected\models\Exam.php(151): CDbCommand->queryRow()
#2 C:\AppServ\www\epretest\protected\views\web\student\view.php(204): Exam->getTotalExamBySubjectId('1')
#3 C:\AppServ\www\epretest\framework\web\CBaseController.php(127): require('C:\AppServ\www\...')
#4 C:\AppServ\www\epretest\framework\web\CBaseController.php(96): CBaseController->renderInternal('C:\AppServ\www\...', Array, true)
#5 C:\AppServ\www\epretest\framework\web\CController.php(870): CBaseController->renderFile('C:\AppServ\www\...', Array, true)
#6 C:\AppServ\www\epretest\framework\web\CController.php(783): CController->renderPartial('view', Array, true)
#7 C:\AppServ\www\epretest\protected\controllers\web\StudentController.php(144): CController->render('view', Array)
#8 C:\AppServ\www\epretest\framework\web\actions\CInlineAction.php(50): StudentController->actionView()
#9 C:\AppServ\www\epretest\framework\web\CController.php(309): CInlineAction->runWithParams(Array)
#10 C:\AppServ\www\epretest\framework\web\filters\CFilterChain.php(134): CController->runAction(Object(CInlineAction))
#11 C:\AppServ\www\epretest\framework\web\filters\CFilter.php(41): CFilterChain->run()
#12 C:\AppServ\www\epretest\framework\web\CController.php(1146): CFilter->filter(Object(CFilterChain))
#13 C:\AppServ\www\epretest\framework\web\filters\CInlineFilter.php(59): CController->filterAccessControl(Object(CFilterChain))
#14 C:\AppServ\www\epretest\framework\web\filters\CFilterChain.php(131): CInlineFilter->filter(Object(CFilterChain))
#15 C:\AppServ\www\epretest\framework\web\CController.php(292): CFilterChain->run()
#16 C:\AppServ\www\epretest\framework\web\CController.php(266): CController->runActionWithFilters(Object(CInlineAction), Array)
#17 C:\AppServ\www\epretest\framework\web\CWebApplication.php(276): CController->run('view')
#18 C:\AppServ\www\epretest\framework\web\CWebApplication.php(135): CWebApplication->runController('student/view')
#19 C:\AppServ\www\epretest\framework\base\CApplication.php(162): CWebApplication->processRequest()
#20 C:\AppServ\www\epretest\protected\components\WebApplicationEndBehavior.php(26): CApplication->run()
#21 [internal function]: WebApplicationEndBehavior->runEnd('web')
#22 C:\AppServ\www\epretest\framework\base\CComponent.php(261): call_user_func_array(Array, Array)
#23 [internal function]: CComponent->__call('runEnd', Array)
#24 C:\AppServ\www\epretest\index.php(11): CWebApplication->runEnd('web')
#25 {main}
REQUEST_URI=/epretest/index.php?r=student/view&level=13
---
2014/05/03 10:01:26 [error] [system.db.CDbCommand] CDbCommand::execute() failed: SQLSTATE[HY000]: General error: 1452 Cannot add or update a child row: a foreign key constraint fails (`epretest/esto_test_record`, CONSTRAINT `esto_test_record_ibfk_1` FOREIGN KEY (`exam_id`) REFERENCES `esto_exam` (`exam_id`)). The SQL statement executed was: INSERT INTO `esto_test_record` (`exam_id`, `student_id`, `score`, `date_attended`, `elapse_time`, `status`) VALUES (:exam_id, :student_id, :score, :date_attended, :elapse_time, :status).
in C:\AppServ\www\epretest\protected\models\TestRecord.php (224)
in C:\AppServ\www\epretest\protected\controllers\web\ExamController.php (192)
in C:\AppServ\www\epretest\protected\components\WebApplicationEndBehavior.php (26)
2014/05/03 10:01:26 [error] [exception.CDbException] exception 'CDbException' with message 'CDbCommand failed to execute the SQL statement: SQLSTATE[HY000]: General error: 1452 Cannot add or update a child row: a foreign key constraint fails (`epretest/esto_test_record`, CONSTRAINT `esto_test_record_ibfk_1` FOREIGN KEY (`exam_id`) REFERENCES `esto_exam` (`exam_id`)). The SQL statement executed was: INSERT INTO `esto_test_record` (`exam_id`, `student_id`, `score`, `date_attended`, `elapse_time`, `status`) VALUES (:exam_id, :student_id, :score, :date_attended, :elapse_time, :status)' in C:\AppServ\www\epretest\framework\db\CDbCommand.php:354
Stack trace:
#0 C:\AppServ\www\epretest\framework\db\CDbCommand.php(1178): CDbCommand->execute(Array)
#1 C:\AppServ\www\epretest\protected\models\TestRecord.php(224): CDbCommand->insert('esto_test_recor...', Array)
#2 C:\AppServ\www\epretest\protected\controllers\web\ExamController.php(192): TestRecord->saveFirstLogTestRecord('undefined', '8')
#3 C:\AppServ\www\epretest\framework\web\actions\CInlineAction.php(50): ExamController->actionUsecredit()
#4 C:\AppServ\www\epretest\framework\web\CController.php(309): CInlineAction->runWithParams(Array)
#5 C:\AppServ\www\epretest\framework\web\CController.php(287): CController->runAction(Object(CInlineAction))
#6 C:\AppServ\www\epretest\framework\web\CController.php(266): CController->runActionWithFilters(Object(CInlineAction), Array)
#7 C:\AppServ\www\epretest\framework\web\CWebApplication.php(276): CController->run('usecredit')
#8 C:\AppServ\www\epretest\framework\web\CWebApplication.php(135): CWebApplication->runController('exam/usecredit')
#9 C:\AppServ\www\epretest\framework\base\CApplication.php(162): CWebApplication->processRequest()
#10 C:\AppServ\www\epretest\protected\components\WebApplicationEndBehavior.php(26): CApplication->run()
#11 [internal function]: WebApplicationEndBehavior->runEnd('web')
#12 C:\AppServ\www\epretest\framework\base\CComponent.php(261): call_user_func_array(Array, Array)
#13 [internal function]: CComponent->__call('runEnd', Array)
#14 C:\AppServ\www\epretest\index.php(11): CWebApplication->runEnd('web')
#15 {main}
REQUEST_URI=/epretest/index.php?r=exam/usecredit&credit=undefined&id=undefined
HTTP_REFERER=http://localhost/epretest/index.php?r=exam&id=32@%3$34$aw8
---
2014/05/03 10:17:24 [error] [php] Invalid argument supplied for foreach() (C:\AppServ\www\epretest\protected\views\web\student\viewall.php:125)
Stack trace:
#0 C:\AppServ\www\epretest\framework\web\CController.php(870): StudentController->renderFile()
#1 C:\AppServ\www\epretest\framework\web\CController.php(783): StudentController->renderPartial()
#2 C:\AppServ\www\epretest\protected\controllers\web\StudentController.php(706): StudentController->render()
#3 C:\AppServ\www\epretest\framework\web\actions\CInlineAction.php(50): StudentController->actionViewAll()
#4 C:\AppServ\www\epretest\framework\web\CController.php(309): CInlineAction->runWithParams()
#5 C:\AppServ\www\epretest\framework\web\filters\CFilterChain.php(134): StudentController->runAction()
#6 C:\AppServ\www\epretest\framework\web\filters\CFilter.php(41): CFilterChain->run()
#7 C:\AppServ\www\epretest\framework\web\CController.php(1146): CAccessControlFilter->filter()
#8 C:\AppServ\www\epretest\framework\web\filters\CInlineFilter.php(59): StudentController->filterAccessControl()
#9 C:\AppServ\www\epretest\framework\web\filters\CFilterChain.php(131): CInlineFilter->filter()
#10 C:\AppServ\www\epretest\framework\web\CController.php(292): CFilterChain->run()
#11 C:\AppServ\www\epretest\framework\web\CController.php(266): StudentController->runActionWithFilters()
#12 C:\AppServ\www\epretest\framework\web\CWebApplication.php(276): StudentController->run()
#13 C:\AppServ\www\epretest\framework\web\CWebApplication.php(135): CWebApplication->runController()
#14 C:\AppServ\www\epretest\framework\base\CApplication.php(162): CWebApplication->processRequest()
#15 C:\AppServ\www\epretest\protected\components\WebApplicationEndBehavior.php(26): CWebApplication->run()
#16 unknown(0): WebApplicationEndBehavior->runEnd()
#17 C:\AppServ\www\epretest\framework\base\CComponent.php(261): call_user_func_array()
#18 unknown(0): CWebApplication->__call()
#19 C:\AppServ\www\epretest\index.php(11): CWebApplication->runEnd()
REQUEST_URI=/epretest/index.php?r=student/viewall
in C:\AppServ\www\epretest\protected\views\web\student\viewall.php (125)
in C:\AppServ\www\epretest\protected\controllers\web\StudentController.php (706)
in C:\AppServ\www\epretest\protected\components\WebApplicationEndBehavior.php (26)
2014/05/04 00:10:58 [warning] [application] Failed to set unsafe attribute "hddnValue" of "LoginForm".
in C:\AppServ\www\epretest\protected\controllers\web\SiteController.php (265)
in C:\AppServ\www\epretest\protected\components\WebApplicationEndBehavior.php (26)
in C:\AppServ\www\epretest\index.php (11)
2014/05/04 00:54:10 [warning] [application] Failed to set unsafe attribute "hddnValue" of "LoginForm".
in C:\AppServ\www\epretest\protected\controllers\web\SiteController.php (265)
in C:\AppServ\www\epretest\protected\components\WebApplicationEndBehavior.php (26)
in C:\AppServ\www\epretest\index.php (11)
2014/05/04 01:54:26 [error] [system.db.CDbCommand] CDbCommand::execute() failed: SQLSTATE[HY000]: General error: 1452 Cannot add or update a child row: a foreign key constraint fails (`epretest/esto_test_record`, CONSTRAINT `esto_test_record_ibfk_1` FOREIGN KEY (`exam_id`) REFERENCES `esto_exam` (`exam_id`)). The SQL statement executed was: INSERT INTO `esto_test_record` (`exam_id`, `student_id`, `score`, `date_attended`, `elapse_time`, `status`) VALUES (:exam_id, :student_id, :score, :date_attended, :elapse_time, :status).
in C:\AppServ\www\epretest\protected\models\TestRecord.php (224)
in C:\AppServ\www\epretest\protected\controllers\web\ExamController.php (192)
in C:\AppServ\www\epretest\protected\components\WebApplicationEndBehavior.php (26)
2014/05/04 01:54:26 [error] [exception.CDbException] exception 'CDbException' with message 'CDbCommand failed to execute the SQL statement: SQLSTATE[HY000]: General error: 1452 Cannot add or update a child row: a foreign key constraint fails (`epretest/esto_test_record`, CONSTRAINT `esto_test_record_ibfk_1` FOREIGN KEY (`exam_id`) REFERENCES `esto_exam` (`exam_id`)). The SQL statement executed was: INSERT INTO `esto_test_record` (`exam_id`, `student_id`, `score`, `date_attended`, `elapse_time`, `status`) VALUES (:exam_id, :student_id, :score, :date_attended, :elapse_time, :status)' in C:\AppServ\www\epretest\framework\db\CDbCommand.php:354
Stack trace:
#0 C:\AppServ\www\epretest\framework\db\CDbCommand.php(1178): CDbCommand->execute(Array)
#1 C:\AppServ\www\epretest\protected\models\TestRecord.php(224): CDbCommand->insert('esto_test_recor...', Array)
#2 C:\AppServ\www\epretest\protected\controllers\web\ExamController.php(192): TestRecord->saveFirstLogTestRecord('undefined', '8')
#3 C:\AppServ\www\epretest\framework\web\actions\CInlineAction.php(50): ExamController->actionUsecredit()
#4 C:\AppServ\www\epretest\framework\web\CController.php(309): CInlineAction->runWithParams(Array)
#5 C:\AppServ\www\epretest\framework\web\CController.php(287): CController->runAction(Object(CInlineAction))
#6 C:\AppServ\www\epretest\framework\web\CController.php(266): CController->runActionWithFilters(Object(CInlineAction), Array)
#7 C:\AppServ\www\epretest\framework\web\CWebApplication.php(276): CController->run('usecredit')
#8 C:\AppServ\www\epretest\framework\web\CWebApplication.php(135): CWebApplication->runController('exam/usecredit')
#9 C:\AppServ\www\epretest\framework\base\CApplication.php(162): CWebApplication->processRequest()
#10 C:\AppServ\www\epretest\protected\components\WebApplicationEndBehavior.php(26): CApplication->run()
#11 [internal function]: WebApplicationEndBehavior->runEnd('web')
#12 C:\AppServ\www\epretest\framework\base\CComponent.php(261): call_user_func_array(Array, Array)
#13 [internal function]: CComponent->__call('runEnd', Array)
#14 C:\AppServ\www\epretest\index.php(11): CWebApplication->runEnd('web')
#15 {main}
REQUEST_URI=/epretest/index.php?r=exam/usecredit&credit=undefined&id=undefined
HTTP_REFERER=http://localhost/epretest/index.php?r=exam&id=32@%3$1$aw8
---
2014/05/04 03:09:54 [error] [system.db.CDbCommand] CDbCommand::execute() failed: SQLSTATE[HY000]: General error: 1452 Cannot add or update a child row: a foreign key constraint fails (`epretest/esto_test_record`, CONSTRAINT `esto_test_record_ibfk_1` FOREIGN KEY (`exam_id`) REFERENCES `esto_exam` (`exam_id`)). The SQL statement executed was: INSERT INTO `esto_test_record` (`exam_id`, `student_id`, `score`, `date_attended`, `elapse_time`, `status`) VALUES (:exam_id, :student_id, :score, :date_attended, :elapse_time, :status).
in C:\AppServ\www\epretest\protected\models\TestRecord.php (224)
in C:\AppServ\www\epretest\protected\controllers\web\ExamController.php (192)
in C:\AppServ\www\epretest\protected\components\WebApplicationEndBehavior.php (26)
2014/05/04 03:09:54 [error] [exception.CDbException] exception 'CDbException' with message 'CDbCommand failed to execute the SQL statement: SQLSTATE[HY000]: General error: 1452 Cannot add or update a child row: a foreign key constraint fails (`epretest/esto_test_record`, CONSTRAINT `esto_test_record_ibfk_1` FOREIGN KEY (`exam_id`) REFERENCES `esto_exam` (`exam_id`)). The SQL statement executed was: INSERT INTO `esto_test_record` (`exam_id`, `student_id`, `score`, `date_attended`, `elapse_time`, `status`) VALUES (:exam_id, :student_id, :score, :date_attended, :elapse_time, :status)' in C:\AppServ\www\epretest\framework\db\CDbCommand.php:354
Stack trace:
#0 C:\AppServ\www\epretest\framework\db\CDbCommand.php(1178): CDbCommand->execute(Array)
#1 C:\AppServ\www\epretest\protected\models\TestRecord.php(224): CDbCommand->insert('esto_test_recor...', Array)
#2 C:\AppServ\www\epretest\protected\controllers\web\ExamController.php(192): TestRecord->saveFirstLogTestRecord('undefined', '8')
#3 C:\AppServ\www\epretest\framework\web\actions\CInlineAction.php(50): ExamController->actionUsecredit()
#4 C:\AppServ\www\epretest\framework\web\CController.php(309): CInlineAction->runWithParams(Array)
#5 C:\AppServ\www\epretest\framework\web\CController.php(287): CController->runAction(Object(CInlineAction))
#6 C:\AppServ\www\epretest\framework\web\CController.php(266): CController->runActionWithFilters(Object(CInlineAction), Array)
#7 C:\AppServ\www\epretest\framework\web\CWebApplication.php(276): CController->run('usecredit')
#8 C:\AppServ\www\epretest\framework\web\CWebApplication.php(135): CWebApplication->runController('exam/usecredit')
#9 C:\AppServ\www\epretest\framework\base\CApplication.php(162): CWebApplication->processRequest()
#10 C:\AppServ\www\epretest\protected\components\WebApplicationEndBehavior.php(26): CApplication->run()
#11 [internal function]: WebApplicationEndBehavior->runEnd('web')
#12 C:\AppServ\www\epretest\framework\base\CComponent.php(261): call_user_func_array(Array, Array)
#13 [internal function]: CComponent->__call('runEnd', Array)
#14 C:\AppServ\www\epretest\index.php(11): CWebApplication->runEnd('web')
#15 {main}
REQUEST_URI=/epretest/index.php?r=exam/usecredit&credit=undefined&id=undefined
HTTP_REFERER=http://localhost/epretest/index.php?r=exam&id=32@%3$16$aw8
---
2014/05/04 03:18:05 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'The requested page does not exist.' in C:\AppServ\www\epretest\protected\controllers\web\ExamController.php:218
Stack trace:
#0 C:\AppServ\www\epretest\protected\controllers\web\ExamController.php(117): ExamController->loadTestRecord(NULL)
#1 C:\AppServ\www\epretest\framework\web\actions\CInlineAction.php(50): ExamController->actionSubmit()
#2 C:\AppServ\www\epretest\framework\web\CController.php(309): CInlineAction->runWithParams(Array)
#3 C:\AppServ\www\epretest\framework\web\CController.php(287): CController->runAction(Object(CInlineAction))
#4 C:\AppServ\www\epretest\framework\web\CController.php(266): CController->runActionWithFilters(Object(CInlineAction), Array)
#5 C:\AppServ\www\epretest\framework\web\CWebApplication.php(276): CController->run('submit')
#6 C:\AppServ\www\epretest\framework\web\CWebApplication.php(135): CWebApplication->runController('exam/submit')
#7 C:\AppServ\www\epretest\framework\base\CApplication.php(162): CWebApplication->processRequest()
#8 C:\AppServ\www\epretest\protected\components\WebApplicationEndBehavior.php(26): CApplication->run()
#9 [internal function]: WebApplicationEndBehavior->runEnd('web')
#10 C:\AppServ\www\epretest\framework\base\CComponent.php(261): call_user_func_array(Array, Array)
#11 [internal function]: CComponent->__call('runEnd', Array)
#12 C:\AppServ\www\epretest\index.php(11): CWebApplication->runEnd('web')
#13 {main}
REQUEST_URI=/epretest/index.php?r=exam/submit&id=29
---
2014/05/04 03:18:08 [error] [system.db.CDbCommand] CDbCommand::execute() failed: SQLSTATE[HY000]: General error: 1452 Cannot add or update a child row: a foreign key constraint fails (`epretest/esto_test_record`, CONSTRAINT `esto_test_record_ibfk_1` FOREIGN KEY (`exam_id`) REFERENCES `esto_exam` (`exam_id`)). The SQL statement executed was: INSERT INTO `esto_test_record` (`exam_id`, `student_id`, `score`, `date_attended`, `elapse_time`, `status`) VALUES (:exam_id, :student_id, :score, :date_attended, :elapse_time, :status).
in C:\AppServ\www\epretest\protected\models\TestRecord.php (224)
in C:\AppServ\www\epretest\protected\controllers\web\ExamController.php (192)
in C:\AppServ\www\epretest\protected\components\WebApplicationEndBehavior.php (26)
2014/05/04 03:18:08 [error] [exception.CDbException] exception 'CDbException' with message 'CDbCommand failed to execute the SQL statement: SQLSTATE[HY000]: General error: 1452 Cannot add or update a child row: a foreign key constraint fails (`epretest/esto_test_record`, CONSTRAINT `esto_test_record_ibfk_1` FOREIGN KEY (`exam_id`) REFERENCES `esto_exam` (`exam_id`)). The SQL statement executed was: INSERT INTO `esto_test_record` (`exam_id`, `student_id`, `score`, `date_attended`, `elapse_time`, `status`) VALUES (:exam_id, :student_id, :score, :date_attended, :elapse_time, :status)' in C:\AppServ\www\epretest\framework\db\CDbCommand.php:354
Stack trace:
#0 C:\AppServ\www\epretest\framework\db\CDbCommand.php(1178): CDbCommand->execute(Array)
#1 C:\AppServ\www\epretest\protected\models\TestRecord.php(224): CDbCommand->insert('esto_test_recor...', Array)
#2 C:\AppServ\www\epretest\protected\controllers\web\ExamController.php(192): TestRecord->saveFirstLogTestRecord('undefined', '8')
#3 C:\AppServ\www\epretest\framework\web\actions\CInlineAction.php(50): ExamController->actionUsecredit()
#4 C:\AppServ\www\epretest\framework\web\CController.php(309): CInlineAction->runWithParams(Array)
#5 C:\AppServ\www\epretest\framework\web\CController.php(287): CController->runAction(Object(CInlineAction))
#6 C:\AppServ\www\epretest\framework\web\CController.php(266): CController->runActionWithFilters(Object(CInlineAction), Array)
#7 C:\AppServ\www\epretest\framework\web\CWebApplication.php(276): CController->run('usecredit')
#8 C:\AppServ\www\epretest\framework\web\CWebApplication.php(135): CWebApplication->runController('exam/usecredit')
#9 C:\AppServ\www\epretest\framework\base\CApplication.php(162): CWebApplication->processRequest()
#10 C:\AppServ\www\epretest\protected\components\WebApplicationEndBehavior.php(26): CApplication->run()
#11 [internal function]: WebApplicationEndBehavior->runEnd('web')
#12 C:\AppServ\www\epretest\framework\base\CComponent.php(261): call_user_func_array(Array, Array)
#13 [internal function]: CComponent->__call('runEnd', Array)
#14 C:\AppServ\www\epretest\index.php(11): CWebApplication->runEnd('web')
#15 {main}
REQUEST_URI=/epretest/index.php?r=exam/usecredit&credit=undefined&id=undefined
HTTP_REFERER=http://localhost/epretest/index.php?r=exam&id=32@%3$29$aw8
---
2014/05/04 11:02:18 [warning] [application] Failed to set unsafe attribute "hddnValue" of "LoginForm".
in C:\AppServ\www\epretest\protected\controllers\web\SiteController.php (265)
in C:\AppServ\www\epretest\protected\components\WebApplicationEndBehavior.php (26)
in C:\AppServ\www\epretest\index.php (11)
2014/05/05 11:18:09 [warning] [application] Failed to set unsafe attribute "hddnValue" of "LoginForm".
in C:\AppServ\www\epretest\protected\controllers\web\SiteController.php (265)
in C:\AppServ\www\epretest\protected\components\WebApplicationEndBehavior.php (26)
in C:\AppServ\www\epretest\index.php (11)
2014/05/05 11:39:03 [warning] [application] Failed to set unsafe attribute "level_id" of "Student".
in C:\AppServ\www\epretest\protected\controllers\admin\StudentController.php (80)
in C:\AppServ\www\epretest\protected\components\WebApplicationEndBehavior.php (26)
in C:\AppServ\www\epretest\admin.php (11)
2014/05/05 11:39:12 [warning] [application] Failed to set unsafe attribute "hddnValue" of "LoginForm".
in C:\AppServ\www\epretest\protected\controllers\web\SiteController.php (265)
in C:\AppServ\www\epretest\protected\components\WebApplicationEndBehavior.php (26)
in C:\AppServ\www\epretest\index.php (11)
2014/05/05 11:45:22 [warning] [application] Failed to set unsafe attribute "hddnValue" of "LoginForm".
in C:\AppServ\www\epretest\protected\controllers\web\SiteController.php (265)
in C:\AppServ\www\epretest\protected\components\WebApplicationEndBehavior.php (26)
in C:\AppServ\www\epretest\index.php (11)
2014/05/05 11:51:40 [warning] [application] Failed to set unsafe attribute "hddnValue" of "LoginForm".
in C:\AppServ\www\epretest\protected\controllers\web\SiteController.php (265)
in C:\AppServ\www\epretest\protected\components\WebApplicationEndBehavior.php (26)
in C:\AppServ\www\epretest\index.php (11)
2014/05/05 12:07:29 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'The requested page does not exist.' in C:\AppServ\www\epretest\protected\controllers\web\ExamController.php:223
Stack trace:
#0 C:\AppServ\www\epretest\protected\controllers\web\ExamController.php(241): ExamController->loadTestRecord(NULL)
#1 C:\AppServ\www\epretest\protected\controllers\web\ExamController.php(108): ExamController->saveExamForm()
#2 C:\AppServ\www\epretest\framework\web\actions\CInlineAction.php(50): ExamController->actionSubmit()
#3 C:\AppServ\www\epretest\framework\web\CController.php(309): CInlineAction->runWithParams(Array)
#4 C:\AppServ\www\epretest\framework\web\CController.php(287): CController->runAction(Object(CInlineAction))
#5 C:\AppServ\www\epretest\framework\web\CController.php(266): CController->runActionWithFilters(Object(CInlineAction), Array)
#6 C:\AppServ\www\epretest\framework\web\CWebApplication.php(276): CController->run('submit')
#7 C:\AppServ\www\epretest\framework\web\CWebApplication.php(135): CWebApplication->runController('exam/submit')
#8 C:\AppServ\www\epretest\framework\base\CApplication.php(162): CWebApplication->processRequest()
#9 C:\AppServ\www\epretest\protected\components\WebApplicationEndBehavior.php(26): CApplication->run()
#10 [internal function]: WebApplicationEndBehavior->runEnd('web')
#11 C:\AppServ\www\epretest\framework\base\CComponent.php(261): call_user_func_array(Array, Array)
#12 [internal function]: CComponent->__call('runEnd', Array)
#13 C:\AppServ\www\epretest\index.php(11): CWebApplication->runEnd('web')
#14 {main}
REQUEST_URI=/epretest/index.php?r=exam/submit&id=35
HTTP_REFERER=http://localhost/epretest/index.php?r=exam&id=32@%3$35$aw8
---
2014/05/05 12:32:11 [warning] [application] Failed to set unsafe attribute "hddnValue" of "LoginForm".
in C:\AppServ\www\epretest\protected\controllers\web\SiteController.php (265)
in C:\AppServ\www\epretest\protected\components\WebApplicationEndBehavior.php (26)
in C:\AppServ\www\epretest\index.php (11)
2014/05/05 17:05:02 [warning] [application] Failed to set unsafe attribute "hddnValue" of "LoginForm".
in C:\AppServ\www\epretest\protected\controllers\web\SiteController.php (265)
in C:\AppServ\www\epretest\protected\components\WebApplicationEndBehavior.php (26)
in C:\AppServ\www\epretest\index.php (11)
2014/05/06 17:04:49 [warning] [application] Failed to set unsafe attribute "hddnValue" of "LoginForm".
in C:\AppServ\www\epretest\protected\controllers\web\SiteController.php (265)
in C:\AppServ\www\epretest\protected\components\WebApplicationEndBehavior.php (26)
in C:\AppServ\www\epretest\index.php (11)
2014/05/07 02:36:43 [warning] [application] Failed to set unsafe attribute "hddnValue" of "LoginForm".
in C:\AppServ\www\epretest\protected\controllers\web\SiteController.php (265)
in C:\AppServ\www\epretest\protected\components\WebApplicationEndBehavior.php (26)
in C:\AppServ\www\epretest\index.php (11)
2014/05/07 10:27:47 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'The requested page does not exist.' in C:\AppServ\www\epretest\protected\controllers\web\ExamController.php:224
Stack trace:
#0 C:\AppServ\www\epretest\protected\controllers\web\ExamController.php(242): ExamController->loadTestRecord(NULL)
#1 C:\AppServ\www\epretest\protected\controllers\web\ExamController.php(108): ExamController->saveExamForm()
#2 C:\AppServ\www\epretest\framework\web\actions\CInlineAction.php(50): ExamController->actionSubmit()
#3 C:\AppServ\www\epretest\framework\web\CController.php(309): CInlineAction->runWithParams(Array)
#4 C:\AppServ\www\epretest\framework\web\CController.php(287): CController->runAction(Object(CInlineAction))
#5 C:\AppServ\www\epretest\framework\web\CController.php(266): CController->runActionWithFilters(Object(CInlineAction), Array)
#6 C:\AppServ\www\epretest\framework\web\CWebApplication.php(276): CController->run('submit')
#7 C:\AppServ\www\epretest\framework\web\CWebApplication.php(135): CWebApplication->runController('exam/submit')
#8 C:\AppServ\www\epretest\framework\base\CApplication.php(162): CWebApplication->processRequest()
#9 C:\AppServ\www\epretest\protected\components\WebApplicationEndBehavior.php(26): CApplication->run()
#10 [internal function]: WebApplicationEndBehavior->runEnd('web')
#11 C:\AppServ\www\epretest\framework\base\CComponent.php(261): call_user_func_array(Array, Array)
#12 [internal function]: CComponent->__call('runEnd', Array)
#13 C:\AppServ\www\epretest\index.php(11): CWebApplication->runEnd('web')
#14 {main}
REQUEST_URI=/epretest/index.php?r=exam/submit&id=37
HTTP_REFERER=http://localhost/epretest/index.php?r=exam&id=32@%3$37$aw8
---
2014/05/07 10:29:17 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'The requested page does not exist.' in C:\AppServ\www\epretest\protected\controllers\web\ExamController.php:224
Stack trace:
#0 C:\AppServ\www\epretest\protected\controllers\web\ExamController.php(242): ExamController->loadTestRecord(NULL)
#1 C:\AppServ\www\epretest\protected\controllers\web\ExamController.php(108): ExamController->saveExamForm()
#2 C:\AppServ\www\epretest\framework\web\actions\CInlineAction.php(50): ExamController->actionSubmit()
#3 C:\AppServ\www\epretest\framework\web\CController.php(309): CInlineAction->runWithParams(Array)
#4 C:\AppServ\www\epretest\framework\web\CController.php(287): CController->runAction(Object(CInlineAction))
#5 C:\AppServ\www\epretest\framework\web\CController.php(266): CController->runActionWithFilters(Object(CInlineAction), Array)
#6 C:\AppServ\www\epretest\framework\web\CWebApplication.php(276): CController->run('submit')
#7 C:\AppServ\www\epretest\framework\web\CWebApplication.php(135): CWebApplication->runController('exam/submit')
#8 C:\AppServ\www\epretest\framework\base\CApplication.php(162): CWebApplication->processRequest()
#9 C:\AppServ\www\epretest\protected\components\WebApplicationEndBehavior.php(26): CApplication->run()
#10 [internal function]: WebApplicationEndBehavior->runEnd('web')
#11 C:\AppServ\www\epretest\framework\base\CComponent.php(261): call_user_func_array(Array, Array)
#12 [internal function]: CComponent->__call('runEnd', Array)
#13 C:\AppServ\www\epretest\index.php(11): CWebApplication->runEnd('web')
#14 {main}
REQUEST_URI=/epretest/index.php?r=exam/submit&id=35
HTTP_REFERER=http://localhost/epretest/index.php?r=exam&id=32@%3$35$aw8
---
2014/05/07 10:31:06 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'The requested page does not exist.' in C:\AppServ\www\epretest\protected\controllers\web\ExamController.php:224
Stack trace:
#0 C:\AppServ\www\epretest\protected\controllers\web\ExamController.php(242): ExamController->loadTestRecord(NULL)
#1 C:\AppServ\www\epretest\protected\controllers\web\ExamController.php(108): ExamController->saveExamForm()
#2 C:\AppServ\www\epretest\framework\web\actions\CInlineAction.php(50): ExamController->actionSubmit()
#3 C:\AppServ\www\epretest\framework\web\CController.php(309): CInlineAction->runWithParams(Array)
#4 C:\AppServ\www\epretest\framework\web\CController.php(287): CController->runAction(Object(CInlineAction))
#5 C:\AppServ\www\epretest\framework\web\CController.php(266): CController->runActionWithFilters(Object(CInlineAction), Array)
#6 C:\AppServ\www\epretest\framework\web\CWebApplication.php(276): CController->run('submit')
#7 C:\AppServ\www\epretest\framework\web\CWebApplication.php(135): CWebApplication->runController('exam/submit')
#8 C:\AppServ\www\epretest\framework\base\CApplication.php(162): CWebApplication->processRequest()
#9 C:\AppServ\www\epretest\protected\components\WebApplicationEndBehavior.php(26): CApplication->run()
#10 [internal function]: WebApplicationEndBehavior->runEnd('web')
#11 C:\AppServ\www\epretest\framework\base\CComponent.php(261): call_user_func_array(Array, Array)
#12 [internal function]: CComponent->__call('runEnd', Array)
#13 C:\AppServ\www\epretest\index.php(11): CWebApplication->runEnd('web')
#14 {main}
REQUEST_URI=/epretest/index.php?r=exam/submit&id=35
HTTP_REFERER=http://localhost/epretest/index.php?r=exam&id=32@%3$35$aw8
---
2014/05/07 11:07:10 [error] [php] Cannot modify header information - headers already sent by (output started at C:\AppServ\www\epretest\framework\web\CController.php:794) (C:\AppServ\www\epretest\framework\web\CHttpRequest.php:763)
Stack trace:
#0 C:\AppServ\www\epretest\protected\controllers\web\ExamController.php(143): ExamController->redirect()
#1 C:\AppServ\www\epretest\framework\web\actions\CInlineAction.php(50): ExamController->actionTimeout()
#2 C:\AppServ\www\epretest\framework\web\CController.php(309): CInlineAction->runWithParams()
#3 C:\AppServ\www\epretest\framework\web\CController.php(287): ExamController->runAction()
#4 C:\AppServ\www\epretest\framework\web\CController.php(266): ExamController->runActionWithFilters()
#5 C:\AppServ\www\epretest\framework\web\CWebApplication.php(276): ExamController->run()
#6 C:\AppServ\www\epretest\framework\web\CWebApplication.php(135): CWebApplication->runController()
#7 C:\AppServ\www\epretest\framework\base\CApplication.php(162): CWebApplication->processRequest()
#8 C:\AppServ\www\epretest\protected\components\WebApplicationEndBehavior.php(26): CWebApplication->run()
#9 unknown(0): WebApplicationEndBehavior->runEnd()
#10 C:\AppServ\www\epretest\framework\base\CComponent.php(261): call_user_func_array()
#11 unknown(0): CWebApplication->__call()
#12 C:\AppServ\www\epretest\index.php(11): CWebApplication->runEnd()
REQUEST_URI=/epretest/index.php?r=exam/timeout&id=38
in C:\AppServ\www\epretest\protected\controllers\web\ExamController.php (143)
in C:\AppServ\www\epretest\protected\components\WebApplicationEndBehavior.php (26)
in C:\AppServ\www\epretest\index.php (11)
2014/05/07 11:59:48 [warning] [application] Failed to set unsafe attribute "hddnValue" of "LoginForm".
in C:\AppServ\www\epretest\protected\controllers\web\SiteController.php (265)
in C:\AppServ\www\epretest\protected\components\WebApplicationEndBehavior.php (26)
in C:\AppServ\www\epretest\index.php (11)
2014/05/07 12:51:41 [warning] [application] Failed to set unsafe attribute "hddnValue" of "LoginForm".
in C:\AppServ\www\epretest\protected\controllers\web\SiteController.php (265)
in C:\AppServ\www\epretest\protected\components\WebApplicationEndBehavior.php (26)
in C:\AppServ\www\epretest\index.php (11)
2014/05/07 13:37:11 [warning] [application] Failed to set unsafe attribute "hddnValue" of "LoginForm".
in C:\AppServ\www\epretest\protected\controllers\web\SiteController.php (265)
in C:\AppServ\www\epretest\protected\components\WebApplicationEndBehavior.php (26)
in C:\AppServ\www\epretest\index.php (11)
